<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="keywords" content="DNA,1953,1997,2006,3' end,3′ end,454 Sequencing,5' end,5′ end,Abstract machine,Adaptation (biology)" />
<link rel="shortcut icon"  />
<link rel="search" type="application/opensearchdescription+xml"  />
<link rel="copyright"  />
		<title>DNA - Wikipedia, the free encyclopedia</title>
		<style type="text/css" media="screen,projection">/*<![CDATA[*/ @import "/skins-1.5/monobook/main.css?9"; /*]]>*/</style>
		<link rel="stylesheet" type="text/css" media="print"  />
		<!--[if lt IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE50Fixes.css";</style><![endif]-->
		<!--[if IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE55Fixes.css";</style><![endif]-->
		<!--[if IE 6]><style type="text/css">@import "/skins-1.5/monobook/IE60Fixes.css";</style><![endif]-->
		<!--[if IE 7]><style type="text/css">@import "/skins-1.5/monobook/IE70Fixes.css?1";</style><![endif]-->
		<!--[if lt IE 7]><script type="text/javascript" src="/skins-1.5/common/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
		
		<script type= "text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "DNA";
			var wgTitle = "DNA";
			var wgArticleId = 7955;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
		                
		<script type="text/javascript" src="/skins-1.5/common/wikibits.js?1"><!-- wikibits js --></script>
		<script type="text/javascript" src="/w/index.php?title=-&amp;action=raw&amp;gen=js"><!-- site js --></script>
		<style type="text/css">/*<![CDATA[*/
@import "/w/index.php?title=MediaWiki:Common.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=MediaWiki:Monobook.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=-&action=raw&gen=css&maxage=2678400";
/*]]>*/</style>
		<!-- Head Scripts -->
		<script type="text/javascript" src="/skins-1.5/common/ajax.js"></script>
	</head>
<body  class="mediawiki ns-0 ltr">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
		<div id="siteNotice"><div style="text-align:right; font-size:80%">Your <b><a  class="extiw" title="wikimedia:Fundraising">continued donations</a></b> keep Wikipedia running!&nbsp;&nbsp;&nbsp;&nbsp;</div>
</div>		<h1 class="firstHeading">DNA</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a >navigation</a>, <a >search</a></div>			<!-- start content -->
			<dl>
<dd><i>For other uses, see <a href="/wiki/DNA_%28disambiguation%29.html" title="DNA (disambiguation)">DNA (disambiguation)</a>.</i></dd>
</dl>
<div class="thumb tright">
<div style="width:272px;"><a  class="internal" title="The general structure of a section of DNA"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/f/f0/DNA_Overview.png/270px-DNA_Overview.png" alt="The general structure of a section of DNA" width="270" height="676" longdesc="/wiki/Image:DNA_Overview.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
The general structure of a section of DNA</div>
</div>
</div>
<p><b>Deoxyribonucleic acid</b> (<b>DNA</b>) is a <a href="/wiki/Nucleic_acid.html" title="Nucleic acid">nucleic acid</a> — usually in the form of a double <a href="/wiki/Helix.html" title="Helix">helix</a> — that contains the <a href="/wiki/Genetics.html" title="Genetics">genetic</a> instructions monitoring the <a href="/wiki/Developmental_biology.html" title="Developmental biology">biological development</a> of all <a href="/wiki/Cell_%28biology%29.html" title="Cell (biology)">cellular</a> forms of <a href="/wiki/Life.html" title="Life">life</a>, and many <a href="/wiki/Virus.html" title="Virus">viruses</a>. DNA is a long <a href="/wiki/Polymer.html" title="Polymer">polymer</a> of <a href="/wiki/Nucleotides.html" title="Nucleotides">nucleotides</a> (a polynucleotide) and encodes the sequence of the <a href="/wiki/Amino_acid_residue.html" title="Amino acid residue">amino acid residues</a> in <a href="/wiki/Protein.html" title="Protein">proteins</a> using the <a href="/wiki/Genetic_code.html" title="Genetic code">genetic code</a>, a <a href="/wiki/Triplet_code.html" title="Triplet code">triplet code</a> of <a href="/wiki/Nucleotide.html" title="Nucleotide">nucleotides</a>. DNA is thought to date back to between approximately 3.5 to 4.6 billion years ago.<sup id="_ref-0" class="reference"><a  title="">[1]</a></sup></p>
<p>In <a href="/wiki/Eukaryote.html" title="Eukaryote">eukaryotic</a> <a href="/wiki/Cell_%28biology%29.html" title="Cell (biology)">cells</a> (which are quite complex), such as those of <a href="/wiki/Plant.html" title="Plant">plants</a>, <a href="/wiki/Animal.html" title="Animal">animals</a>, <a href="/wiki/Fungi.html" title="Fungi">fungi</a> and <a href="/wiki/Protist.html" title="Protist">protists</a>, most of the DNA is located in the <a href="/wiki/Cell_nucleus.html" title="Cell nucleus">cell nucleus</a>. By contrast, in simpler cells called <a href="/wiki/Prokaryotes.html" title="Prokaryotes">prokaryotes</a>, including the <a href="/wiki/Bacterium.html" title="Bacterium">eubacteria</a> and <a href="/wiki/Archaea.html" title="Archaea">archaea</a>, DNA is not separated from the <a href="/wiki/Cytoplasm.html" title="Cytoplasm">cytoplasm</a> by a <a href="/wiki/Nuclear_envelope.html" title="Nuclear envelope">nuclear envelope</a>. The cellular <a href="/wiki/Organelle.html" title="Organelle">organelles</a> known as <a href="/wiki/Chloroplast.html" title="Chloroplast">chloroplasts</a> and <a href="/wiki/Mitochondria.html" title="Mitochondria">mitochondria</a> also carry DNA.</p>
<p>DNA is often referred to as the molecule of <a href="/wiki/Heredity.html" title="Heredity">heredity</a> as it is responsible for the genetic propagation of most <a href="/wiki/Biological_inheritance.html" title="Biological inheritance">inherited</a> <a href="/wiki/Trait_%28biological%29.html" title="Trait (biological)">traits</a>. In humans, these traits range from hair color to disease susceptibility. During <a href="/wiki/Cell_division.html" title="Cell division">cell division</a>, DNA is <a href="/wiki/DNA_replication.html" title="DNA replication">replicated</a> and can be transmitted to offspring during <a href="/wiki/Reproduction.html" title="Reproduction">reproduction</a>. In sexual reproduction, <a href="/wiki/Kinship_and_descent.html" title="Kinship and descent">lineage</a> studies can be done because <a href="/wiki/Mitochondrial_DNA.html" title="Mitochondrial DNA">mitochondrial DNA</a> only comes from the mother, and the <a href="/wiki/Y_chromosome.html" title="Y chromosome">Y chromosome</a> only comes from the father.</p>
<p>Every person's DNA, their genome, is a combination of the DNAs of his parents. The mother's <a href="/wiki/Mitochondrial_DNA.html" title="Mitochondrial DNA">mitochondrial DNA</a> together with 23 <a href="/wiki/Chromosome.html" title="Chromosome">chromosomes</a> from each parent combine to form the genome of a <a href="/wiki/Zygote.html" title="Zygote">zygote</a>, the <a href="/wiki/Fertilization.html" title="Fertilization">fertilized</a> <a href="/wiki/Ovum.html" title="Ovum">egg</a>. As a result, with certain exceptions such as <a href="/wiki/Red_blood_cell.html" title="Red blood cell">red blood cells</a>, most human cells contain 23 pairs of chromosomes, together with mitochondrial DNA inherited from the mother.</p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a ><span class="tocnumber">1</span> <span class="toctext">Overview</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">2</span> <span class="toctext">Molecular structure</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">3</span> <span class="toctext">Nucleotide sequence</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">4</span> <span class="toctext">Replication</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">5</span> <span class="toctext">Mechanical biological properties</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">5.1</span> <span class="toctext">Strand association and dissociation</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">5.2</span> <span class="toctext">Circular DNA</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">5.3</span> <span class="toctext">Long, narrow shape</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">5.4</span> <span class="toctext">Entropic stretching behavior</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">5.5</span> <span class="toctext">Helix geometries</span></a>
<ul>
<li class="toclevel-3"><a ><span class="tocnumber">5.5.1</span> <span class="toctext">Supercoiled DNA</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">5.5.2</span> <span class="toctext">Sugar pucker</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">5.5.3</span> <span class="toctext">A and Z helices formation</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">5.5.4</span> <span class="toctext">Properties of different helical forms</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a ><span class="tocnumber">5.6</span> <span class="toctext">Non-helical forms</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">6</span> <span class="toctext">Strand direction</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">7</span> <span class="toctext">Single-stranded DNA (ssDNA) and repair of mutations</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">8</span> <span class="toctext">History of DNA research</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">8.1</span> <span class="toctext">First isolation of DNA</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">8.2</span> <span class="toctext">Chromosomes and inherited traits</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">8.3</span> <span class="toctext">Discovery of the structure of DNA</span></a>
<ul>
<li class="toclevel-3"><a ><span class="tocnumber">8.3.1</span> <span class="toctext">Helix structure</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">8.3.2</span> <span class="toctext">Complementary nucleotides</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">8.3.3</span> <span class="toctext">Watson and Crick's model</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">8.3.4</span> <span class="toctext">"Central Dogma"</span></a></li>
</ul>
</li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">9</span> <span class="toctext">DNA in practice</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">9.1</span> <span class="toctext">DNA in crime</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">9.2</span> <span class="toctext">DNA in computation</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">9.3</span> <span class="toctext">DNA in historical and anthropological study</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">10</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">11</span> <span class="toctext">External links</span></a></li>
</ul>
</td>
</tr>
</table>
<p><script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script></p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Overview">edit</a>]</div>
<p><a name="Overview" id="Overview"></a></p>
<h2>Overview</h2>
<div class="thumb tright">
<div style="width:183px;"><a  class="internal" title="Animation of a section of DNA rotating."><img src="http://upload.wikimedia.org/wikipedia/commons/8/81/ADN_animation.gif" alt="Animation of a section of DNA rotating." width="181" height="313" longdesc="/wiki/Image:ADN_animation.gif" /></a>
<div class="thumbcaption">Animation of a section of DNA rotating.</div>
</div>
</div>
<div class="thumb tright">
<div style="width:302px;"><a  class="internal" title="DNA base pairing"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/a/a1/Dna_pairing_aa.gif/300px-Dna_pairing_aa.gif" alt="DNA base pairing" width="300" height="257" longdesc="/wiki/Image:Dna_pairing_aa.gif" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
DNA base pairing</div>
</div>
</div>
<p>DNA consists of a pair of <a href="/wiki/Molecule.html" title="Molecule">molecules</a>, organized as strands running start-to-end and joined by <a href="/wiki/Hydrogen_bond.html" title="Hydrogen bond">hydrogen bonds</a> along their lengths.<sup id="_ref-Butler_0" class="reference"><a  title="">[2]</a></sup> Each strand is a chain of chemical "building blocks", called <a href="/wiki/Nucleotide.html" title="Nucleotide">nucleotides</a>, of which there are four types: <a href="/wiki/Adenine.html" title="Adenine">adenine</a> (abbreviated A), <a href="/wiki/Cytosine.html" title="Cytosine">cytosine</a> (C), <a href="/wiki/Guanine.html" title="Guanine">guanine</a> (G) and <a href="/wiki/Thymine.html" title="Thymine">thymine</a> (T).<sup id="_ref-Butler_1" class="reference"><a  title="">[2]</a></sup> (Thymine should not be confused with <a href="/wiki/Thiamine.html" title="Thiamine">thiamine</a>, which is vitamin B<sub>1</sub>.) The DNA of some organisms, most notably of the PBS1 <a href="/wiki/Phage.html" title="Phage">phage</a>, have <a href="/wiki/Uracil.html" title="Uracil">Uracil</a> (U) instead of T.<sup id="_ref-nature1963-takahashi_0" class="reference"><a  title="">[3]</a></sup> These allowable base components of nucleic acids can be arranged in the <a href="/wiki/Polymer.html" title="Polymer">polymer</a> in any order, giving the molecules a high degree of uniqueness.</p>
<p>DNA contains the genetic <a href="/wiki/Information.html" title="Information">information</a> that is inherited by the offspring of an organism; this information is determined by the <a href="/wiki/DNA_sequence.html" title="DNA sequence">sequence</a> of base pairs along its length. A strand of DNA contains <a href="/wiki/Gene.html" title="Gene">genes</a>, areas that <a href="/wiki/Gene_regulation.html" title="Gene regulation">regulate genes</a>, and areas that either have no function, or a function <a href="/wiki/Junk_DNA.html" title="Junk DNA">yet unknown</a>. Genes can be loosely viewed as the organism's "cookbook" or "blueprint".</p>
<p>Each base on one strand forms a bond with just one kind of base on another strand, called a "complementary" base: A bonds with T and C bonds with G. Therefore, the whole double-strand sequence can be described by the sequence on one of the strands, chosen by convention.<sup id="_ref-Butler_2" class="reference"><a  title="">[2]</a></sup> Two nucleotides paired together are called a <a href="/wiki/Base_pair.html" title="Base pair">base pair</a>. On rare occasions, wrong pairing can happen, when <a href="/wiki/Thymine.html" title="Thymine">thymine</a> goes into its <a href="/wiki/Enol.html" title="Enol">enol</a> form or <a href="/wiki/Cytosine.html" title="Cytosine">cytosine</a> goes into its <a href="/wiki/Imino.html" title="Imino">imino</a> form.</p>
<p>The double-stranded structure of DNA provides a simple mechanism for <a href="/wiki/DNA_replication.html" title="DNA replication">DNA replication</a>: the two strands are separated, and then each strand's complement is recreated by exposing the strand to a mixture of the four bases. An enzyme makes the complement strand by finding the correct base in the mixture and bonding it with the original strand. In this way, the base on the old strand dictates which base appears on the new strand, and the cell ends up with an extra copy of its DNA.</p>
<p>Other interesting points:</p>
<ul>
<li>DNA is an acid because of the phosphate groups between each deoxyribose. This is the primary reason why DNA has a negative charge.</li>
<li>The "polarity" of each pair is important: A+T is not the same as T+A, just as C+G is not the same as G+C (note that "polarity" as such is never used in this context -- it's just a suggestive way to get the idea across).</li>
<li><a href="/wiki/Mutation.html" title="Mutation">Mutations</a> are the results of the cells' attempts to repair chemical imperfections in this process, where a base is accidentally skipped, inserted, or incorrectly copied, or the chain is trimmed, or added to. Many mutations can be described as combinations of these accidental "operations". Mutations can also occur after chemical damage (through <a href="/wiki/Mutagens.html" title="Mutagens">mutagens</a>), light (<a href="/wiki/Ultraviolet.html" title="Ultraviolet">UV</a> damage), or through other more complicated gene swapping events.</li>
<li><a href="/wiki/Deoxyribozyme.html" title="Deoxyribozyme">DNA molecules that act as enzymes</a> are known in laboratories, but none have been known to be found in life so far.</li>
<li>In addition to the traditionally viewed duplex form of DNA, DNA can also acquire triplex and quadruplex forms. They have <a href="/wiki/Hoogsteen_base_pair.html" title="Hoogsteen base pair">Hoogsteen base pairing</a> instead of the Watson-Crick base pairing found in duplex forms.</li>
<li>DNA differs from <a href="/wiki/Ribonucleic_acid.html" title="Ribonucleic acid">ribonucleic acid</a> (RNA) by having a sugar 2-deoxyribose instead of <a href="/wiki/Ribose.html" title="Ribose">ribose</a> in its backbone. This is the basic chemical distinction between RNA and DNA. In addition, in most<sup title="The text in the vicinity of this tag needs citation." class="noprint">[<a  title="Wikipedia:Citing sources"><i>citation&#160;needed</i></a>]</sup> RNA, the nucleotides <a href="/wiki/Thymine.html" title="Thymine">thymine</a> (T) are replaced by <a href="/wiki/Uracil.html" title="Uracil">uracil</a> (U).</li>
</ul>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Molecular structure">edit</a>]</div>
<p><a name="Molecular_structure" id="Molecular_structure"></a></p>
<h2>Molecular structure</h2>
<div class="thumb tright">
<div style="width:402px;"><a  class="internal" title="Comparisons between DNA and single stranded RNA with the diagram of the bases showing."><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/0/04/NA-comparedto-DNA_thymineAndUracilCorrected.png/400px-NA-comparedto-DNA_thymineAndUracilCorrected.png" alt="Comparisons between DNA and single stranded RNA with the diagram of the bases showing." width="400" height="406" longdesc="/wiki/Image:NA-comparedto-DNA_thymineAndUracilCorrected.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Comparisons between DNA and single stranded RNA with the diagram of the bases showing.</div>
</div>
</div>
<p>Although sometimes called "the molecule of heredity", DNA macromolecules as people typically think of them are not single molecules. Rather, they are pairs of molecules, which entwine like vines, in the shape of a <b>double <a href="/wiki/Helix.html" title="Helix">helix</a></b> (see the illustration at the right).</p>
<p>Each molecule is a strand of DNA: <b>a chemically linked chain of <a href="/wiki/Nucleotide.html" title="Nucleotide">nucleotides</a>, each of which consists of a <a href="/wiki/Sugar.html" title="Sugar">sugar</a> (<a href="/wiki/Deoxyribose.html" title="Deoxyribose">deoxyribose</a>), a <a href="/wiki/Phosphate.html" title="Phosphate">phosphate</a> and one of five kinds of <a href="/wiki/Nucleobase.html" title="Nucleobase">nucleobases</a> ("bases")</b>. Because DNA strands are composed of these nucleotide subunits, they are <a href="/wiki/Polymer.html" title="Polymer">polymers</a>.</p>
<p>The diversity of the bases means that there are five kinds of nucleotides, which are commonly referred to by the identity of their bases. These are <a href="/wiki/Adenine.html" title="Adenine">adenine</a> (A), <a href="/wiki/Thymine.html" title="Thymine">thymine</a> (T), <a href="/wiki/Uracil.html" title="Uracil">uracil</a> (U), <a href="/wiki/Cytosine.html" title="Cytosine">cytosine</a> (C), and <a href="/wiki/Guanine.html" title="Guanine">guanine</a> (G). U is rarely found in DNA except as a result of chemical degradation of C, but the DNA of some viruses, notably PBS1 phage DNA, has U and not T. Similarly, RNA usually contains U in place of T, but in certain RNAs such as <a href="/wiki/Transfer_RNA.html" title="Transfer RNA">transfer RNA</a>, T is always found in some positions. Thus, the only true difference between DNA and RNA is the sugar, 2-deoxyribose in DNA and ribose in RNA.</p>
<p>In a DNA double helix, two polynucleotide strands can associate through the <a href="/wiki/Hydrophobic_effect.html" title="Hydrophobic effect">hydrophobic effect</a> and <a href="/wiki/Pi_stacking.html" title="Pi stacking">pi stacking</a>. Specificity of which strands stay associated is determined by <a href="/wiki/Base_pair.html" title="Base pair">complementary pairing</a>. Each base forms <a href="/wiki/Hydrogen_bond.html" title="Hydrogen bond">hydrogen bonds</a> readily to only one other base, A to T forming two hydrogen bonds and C to G forming three hydrogen bonds. The GC content and length of each DNA molcule dictates the strength of the association; the more complementary bases exist, the stronger and longer-lasting the association, characterised by the temperature required to break the hydrogen bond, its <a href="/wiki/Melting_temperature.html" title="Melting temperature">Melting temperature</a> (also called <i>T<sub>m</sub></i> value)).</p>
<div class="thumb tleft">
<div style="width:402px;"><a  class="internal" title="The chemical structure of DNA"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/f/f7/DNA_chemical_structure.png/400px-DNA_chemical_structure.png" alt="The chemical structure of DNA" width="400" height="430" longdesc="/wiki/Image:DNA_chemical_structure.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
The chemical structure of DNA</div>
</div>
</div>
<p>A cell's machinery separates the DNA double helix, and uses each DNA strand as a template for synthesizing a new strand which is nearly identical to the previous strand. Errors that occur in the synthesis are called <a href="/wiki/Mutations.html" title="Mutations">mutations</a>. This process of replication is mimiced <a href="/wiki/In_vitro.html" title="In vitro">in vitro</a> by a process called <a href="/wiki/Polymerase_chain_reaction.html" title="Polymerase chain reaction">Polymerase chain reaction</a> (PCR).</p>
<p>Because pairing causes the nucleotide bases to face the helical axis, the sugar and phosphate groups of the nucleotides run along the outside; the two chains they form are sometimes called the "<b>backbones</b>" of the helix. In fact, it is chemical bonds between the phosphates and the sugars that link one nucleotide to the next in the DNA strand.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Nucleotide sequence">edit</a>]</div>
<p><a name="Nucleotide_sequence" id="Nucleotide_sequence"></a></p>
<h2>Nucleotide sequence</h2>
<p>Within a gene, the sequence of <a href="/wiki/Nucleotides.html" title="Nucleotides">nucleotides</a> along a DNA strand defines a messenger RNA sequence which then defines a <a href="/wiki/Protein.html" title="Protein">protein</a>, that an <a href="/wiki/Organism.html" title="Organism">organism</a> is liable to manufacture or "<a href="/wiki/Gene_expression.html" title="Gene expression">express</a>" at one or several points in its life using the information of the sequence. The relationship between the nucleotide sequence and the <a href="/wiki/Amino_acid.html" title="Amino acid">amino-acid</a> sequence of the protein is determined by simple cellular rules of <a href="/wiki/Translation_%28genetics%29.html" title="Translation (genetics)">translation</a>, known collectively as the <a href="/wiki/Genetic_code.html" title="Genetic code">genetic code</a>. The genetic code consists of three-letter 'words' (termed a codon) formed from a sequence of three nucleotides (e.g. ACT, CAG, TTT). These codons can then be translated with <a href="/wiki/Messenger_RNA.html" title="Messenger RNA">messenger RNA</a> and then <a href="/wiki/Transfer_RNA.html" title="Transfer RNA">transfer RNA</a>, with a codon corresponding to a particular amino acid. There are 64 possible codons (4 bases in 3 places <span class="texhtml">4<sup>3</sup></span>) that encode 20 amino acids. Most amino acids, therefore, have more than one possible codon. There are also three 'stop' or 'nonsense' codons signifying the end of the coding region, namely the UAA, UGA and UAG codons.</p>
<p>In many <a href="/wiki/Species.html" title="Species">species</a>, only a small fraction of the total sequence of the <a href="/wiki/Genome.html" title="Genome">genome</a> appears to encode protein. For example, only about 1.5% of the <a href="/wiki/Human_genome.html" title="Human genome">human genome</a> consists of protein-coding <a href="/wiki/Exons.html" title="Exons">exons</a>. The function of the rest is a matter of speculation. It is known that certain nucleotide sequences specify affinity for <a href="/wiki/DNA_binding_protein.html" title="DNA binding protein">DNA binding proteins</a>, which play a wide variety of vital roles, in particular through control of replication and transcription. These sequences are frequently called <a href="/wiki/Regulatory_sequence.html" title="Regulatory sequence">regulatory sequences</a>, and researchers assume that so far they have identified only a tiny fraction of the total that exist. "<a href="/wiki/Junk_DNA.html" title="Junk DNA">Junk DNA</a>" represents sequences that do not yet appear to contain genes or to have a function. The reasons for the presence of so much <a href="/wiki/Non-coding_DNA.html" title="Non-coding DNA">non-coding DNA</a> in <a href="/wiki/Eukaryotic.html" title="Eukaryotic">eukaryotic</a> genomes and the extraordinary differences in <a href="/wiki/Genome_size.html" title="Genome size">genome size</a> ("<a href="/wiki/C-value.html" title="C-value">C-value</a>") among species represent a long-standing puzzle in DNA research known as the "<a href="/wiki/C-value_enigma.html" title="C-value enigma">C-value enigma</a>".</p>
<p>Some DNA sequences play structural roles in chromosomes. <a href="/wiki/Telomere.html" title="Telomere">Telomeres</a> and <a href="/wiki/Centromere.html" title="Centromere">centromeres</a> typically contain few (if any) protein-coding genes, but are important for the function and stability of chromosomes. Some genes code for "RNA genes" (see <a href="/wiki/TRNA.html" title="TRNA">tRNA</a> and <a href="/wiki/RRNA.html" title="RRNA">rRNA</a>). Some RNA genes code for transcripts that function as regulatory RNAs (see <a href="/wiki/RNA_interference.html" title="RNA interference">siRNA</a>) that influence the function of other RNA molecules. The intron-exon structure of some genes (such as immunoglobin and protocadeherin genes) is important for allowing alternative splicing of pre-mRNA which allows several different proteins to be made from the same gene. Some non-coding DNA represents <a href="/wiki/Pseudogene.html" title="Pseudogene">pseudogenes</a>, which have been hypothesized to serve as raw genetic material for the creation of new genes through the process of <a href="/wiki/Gene_duplication.html" title="Gene duplication">gene duplication</a> and <a href="/wiki/Divergent_evolution.html" title="Divergent evolution">divergence</a>. Some non-coding DNA provided hot-spots for duplication of short DNA regions; such sequence duplication has been the major form of genetic change in the human lineage (see evidence from the <a href="/wiki/Chimpanzee_Genome_Project.html" title="Chimpanzee Genome Project">Chimpanzee Genome Project</a>). Exons interspersed with introns allows for "exon shuffling" and the creation of modified genes that might have new adaptive functions. Large amounts of non-coding DNA is probably adaptive in that it provides chromosomal regions where <a href="/wiki/Genetic_recombination.html" title="Genetic recombination">recombination</a> between homologous portions of chromosomes can take place without disrupting the function of genes. Some biologists such as <a href="/wiki/Stuart_Kauffman.html" title="Stuart Kauffman">Stuart Kauffman</a> have speculated that non-coding DNA may modify the rate of evolution of a species.<sup title="The text in the vicinity of this tag needs citation." class="noprint">[<a  title="Wikipedia:Citing sources"><i>citation&#160;needed</i></a>]</sup></p>
<p>Sequence also determines a DNA segment's susceptibility to cleavage by <a href="/wiki/Restriction_enzyme.html" title="Restriction enzyme">restriction enzymes</a>, the quintessential tools of <a href="/wiki/Genetic_engineering.html" title="Genetic engineering">genetic engineering</a>. The position of cleavage sites throughout an individual's genome determines one kind of an individual's "<a href="/wiki/DNA_fingerprinting.html" title="DNA fingerprinting">DNA fingerprint</a>".</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Replication">edit</a>]</div>
<p><a name="Replication" id="Replication"></a></p>
<h2>Replication</h2>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/DNA_replication.html" title="DNA replication">DNA replication</a></i></div>
</dd>
</dl>
<div class="thumb tright">
<div style="width:204px;"><a  class="internal" title="DNA replication"><img src="http://upload.wikimedia.org/wikipedia/commons/0/08/Dna-split.png" alt="DNA replication" width="202" height="397" longdesc="/wiki/Image:Dna-split.png" /></a>
<div class="thumbcaption">DNA replication</div>
</div>
</div>
<p>DNA replication or DNA synthesis is the process of copying the double-stranded DNA prior to <a href="/wiki/Cell_division.html" title="Cell division">cell division</a>. The two resulting double strands are generally almost perfectly identical, but occasionally errors in replication or exposure to chemicals, or radiation can result in a less than perfect copy (see <a href="/wiki/Mutation.html" title="Mutation">mutation</a>), and each of them consists of one original and one newly synthesized strand. This is called <i><a href="/wiki/Semiconservative_replication.html" title="Semiconservative replication">semiconservative replication</a></i>. The process of replication consists of three steps: <i>initiation</i>, <i>elongation</i> and <i>termination</i>. Several methods of <a href="/wiki/DNA_sequencing.html" title="DNA sequencing">DNA sequencing</a> use the characteristics of DNA synthesis to determine the base pair arrangement, most notable are <a href="/wiki/Pyrosequencing.html" title="Pyrosequencing">Pyrosequencing</a> and <a href="/wiki/454_Sequencing.html" title="454 Sequencing">454 Sequencing</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Mechanical biological properties">edit</a>]</div>
<p><a name="Mechanical_biological_properties" id="Mechanical_biological_properties"></a></p>
<h2>Mechanical biological properties</h2>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/Mechanical_properties_of_DNA.html" title="Mechanical properties of DNA">Mechanical properties of DNA</a></i></div>
</dd>
</dl>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Strand association and dissociation">edit</a>]</div>
<p><a name="Strand_association_and_dissociation" id="Strand_association_and_dissociation"></a></p>
<h3>Strand association and dissociation</h3>
<p>The hydrogen bonds between the strands of the double helix are weak enough that they can be easily separated by <a href="/wiki/Enzyme.html" title="Enzyme">enzymes</a>. Enzymes known as <a href="/wiki/Helicase.html" title="Helicase">helicases</a> unwind the strands to facilitate the advance of sequence-reading enzymes such as <a href="/wiki/DNA_polymerase.html" title="DNA polymerase">DNA polymerase</a>. The unwinding requires that helicases chemically cleave the phosphate backbone of one of the strands so that it can swivel around the other. The strands can also be separated by gentle heating, as used in <a href="/wiki/PCR.html" title="PCR">PCR</a>, provided they have fewer than about 10,000 <b>base pairs</b> (10 kilobase pairs, or 10 kbp). The intertwining of the DNA strands makes long segments difficult to separate.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Circular DNA">edit</a>]</div>
<p><a name="Circular_DNA" id="Circular_DNA"></a></p>
<h3>Circular DNA</h3>
<p>When the ends of a piece of double-helical DNA are joined so that it forms a circle, as in <a href="/wiki/Plasmid.html" title="Plasmid">plasmid</a> DNA, the strands are <a href="/wiki/Knot_theory.html" title="Knot theory">topologically</a> knotted. This means they cannot be separated by gentle heating or by any process that does not involve breaking a strand. The task of unknotting topologically linked strands of DNA falls to enzymes known as <a href="/wiki/Topoisomerase.html" title="Topoisomerase">topoisomerases</a>. Some of these enzymes unknot circular DNA by cleaving two strands so that another double-stranded segment can pass through. Unknotting is required for the replication of circular DNA as well as for various types of <a href="/wiki/Recombination.html" title="Recombination">recombination</a> in linear DNA.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Long, narrow shape">edit</a>]</div>
<p><a name="Long.2C_narrow_shape" id="Long.2C_narrow_shape"></a></p>
<h3>Long, narrow shape</h3>
<p>The narrow breadth of the double helix makes it impossible to detect by conventional <a href="/wiki/Transmission_electron_microscope.html" title="Transmission electron microscope">electron microscopy</a>, except by heavy staining. At the same time, the DNA found in many cells can be macroscopic in length -- approximately 2 <a href="/wiki/Meter.html" title="Meter">meters</a> long for strands in a human chromosome.<sup id="_ref-1" class="reference"><a  title="">[4]</a></sup> Consequently, cells must compact or "package" DNA to carry it within them. This is one of the functions of the chromosomes, which contain spool-like <a href="/wiki/Protein.html" title="Protein">proteins</a> known as <a href="/wiki/Histone.html" title="Histone">histones</a>, around which DNA winds.</p>
<div class="thumb tright">
<div style="width:252px;"><a  class="internal" title="DNA Under an electron microscope"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/c/ca/DNA_Under_electron_microscope_Image_3576B-PH.jpg/250px-DNA_Under_electron_microscope_Image_3576B-PH.jpg" alt="DNA Under an electron microscope" width="250" height="171" longdesc="/wiki/Image:DNA_Under_electron_microscope_Image_3576B-PH.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
DNA Under an electron microscope</div>
</div>
</div>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Entropic stretching behavior">edit</a>]</div>
<p><a name="Entropic_stretching_behavior" id="Entropic_stretching_behavior"></a></p>
<h3>Entropic stretching behavior</h3>
<p>When DNA is in solution, it undergoes conformational fluctuations due to the energy available in the <a href="/wiki/Thermal_bath.html" title="Thermal bath">thermal bath</a>. For <a href="/wiki/Entropy.html" title="Entropy">entropic</a> reasons, floppy states are more thermally accessible than stretched out states; for this reason, a single molecule of DNA stretches similarly to a rubber band. Using <a href="/wiki/Optical_tweezers.html" title="Optical tweezers">optical tweezers</a>, the entropic stretching behavior of DNA has been studied and analyzed from a <a href="/wiki/Polymer_physics.html" title="Polymer physics">polymer physics</a> perspective, and it has been found that DNA behaves like the <i>Kratky-Porod</i> <a href="/wiki/Worm-like_chain.html" title="Worm-like chain">worm-like chain</a> model with a persistence length of about 53 nm.</p>
<p>Furthermore, DNA undergoes a stretching <a href="/wiki/Phase_transition.html" title="Phase transition">phase transition</a> at a force of 65 <a href="/wiki/Newton.html" title="Newton">pN</a>; above this force, DNA is thought to take the form that <a href="/wiki/Linus_Pauling.html" title="Linus Pauling">Linus Pauling</a> originally hypothesized, with the phosphates in the middle and bases splayed outward. This proposed structure for overstretched DNA has been called "P-form DNA," in honor of Pauling.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Helix geometries">edit</a>]</div>
<p><a name="Helix_geometries" id="Helix_geometries"></a></p>
<h3>Helix geometries</h3>
<p>The DNA helix can assume one of three slightly different geometries, of which the "B" form described by <a href="/wiki/James_D._Watson.html" title="James D. Watson">James D. Watson</a> and <a href="/wiki/Francis_Crick.html" title="Francis Crick">Francis Crick</a> is believed to predominate in cells. It is 2 <a href="/wiki/Nanometre.html" title="Nanometre">nanometres</a> wide and extends 3.4 nanometres per 10 <a href="/wiki/Base_pair.html" title="Base pair">bp</a> of sequence. This is also the approximate length of sequence in which the double helix makes one complete turn about its axis. This frequency of twist (known as the helical <i>pitch</i>) depends largely on stacking forces that each base exerts on its neighbors in the chain.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Supercoiled DNA">edit</a>]</div>
<p><a name="Supercoiled_DNA" id="Supercoiled_DNA"></a></p>
<h4>Supercoiled DNA</h4>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/Supercoil.html" title="Supercoil">Supercoil</a></i></div>
</dd>
</dl>
<p>The B form of the DNA helix twists 360° per 10 bp in the absence of strain. But many molecular biological processes can induce strain. A DNA segment with excess or insufficient helical twisting is referred to, respectively, as positively or negatively "supercoiled". DNA <i>in vivo</i> is typically negatively supercoiled, which facilitates the unwinding of the double-helix required for <a href="/wiki/Transcription_%28genetics%29.html" title="Transcription (genetics)">RNA transcription</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Sugar pucker">edit</a>]</div>
<p><a name="Sugar_pucker" id="Sugar_pucker"></a></p>
<h4>Sugar pucker</h4>
<p>There are four conformations that the <a href="/wiki/Ribofuranose.html" title="Ribofuranose">ribofuranose</a> rings in nucleotides can acquire:</p>
<ol>
<li>C-2' endo</li>
<li>C-2' exo</li>
<li>C-3' endo</li>
<li>C-3' exo</li>
</ol>
<p>Ribose is usually in C-3'endo, while deoxyribose is usually in the C-2' endo sugar pucker conformation. The A and B forms differ mainly in their <i>sugar pucker</i>. In the A form, the C3' configuration is above the sugar ring, whilst the C2' configuration is below it. Thus, the A form is described as "C3'-endo." Likewise, in the B form, the C2' configuration is above the sugar ring, whilst C3' is below; this is called "C2'-endo." Altered sugar puckering in A-DNA results in shortening the distance between adjacent phosphates by around one angstrom. This gives 11 to 12 base pairs to each helix in the DNA strand, instead of 10.5 in B-DNA. Sugar pucker gives uniform ribbon shape to DNA, a cylindrical open core, and also a deep major groove more narrow and pronounced that grooves found in B-DNA.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: A and Z helices formation">edit</a>]</div>
<p><a name="A_and_Z_helices_formation" id="A_and_Z_helices_formation"></a></p>
<h4>A and Z helices formation</h4>
<p>Examples of two other known double-helical forms of DNA, which differ in their geometry and dimensions, are called A- and <a href="/wiki/Z-DNA.html" title="Z-DNA">Z-DNA</a>. The A form appears likely to occur only in dehydrated (or high salt) samples of DNA<sup title="The text in the vicinity of this tag needs citation." class="noprint">[<a  title="Wikipedia:Citing sources"><i>citation&#160;needed</i></a>]</sup>, such as those used in <a href="/wiki/Crystallography.html" title="Crystallography">crystallographic</a> experiments, and possibly in hybrid pairings of DNA and <a href="/wiki/RNA.html" title="RNA">RNA</a> strands. Segments of DNA that cells have <a href="/wiki/Methylation.html" title="Methylation">methylated</a> may adopt the <a href="/wiki/Z-DNA.html" title="Z-DNA">Z geometry</a> (thought to be for regulatory purposes<sup title="The text in the vicinity of this tag needs citation." class="noprint">[<a  title="Wikipedia:Citing sources"><i>citation&#160;needed</i></a>]</sup>), in which the strands turn about the helical axis in a left-handed and zig-zag pattern (thus the name).</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Properties of different helical forms">edit</a>]</div>
<p><a name="Properties_of_different_helical_forms" id="Properties_of_different_helical_forms"></a></p>
<h4>Properties of different helical forms</h4>
<table border="0" align="center" style="width:450px; border: 1px solid #999; background-color:#FFFFFF">
<tr align="center" bgcolor="#CCCCCC">
<th>Geometry attribute</th>
<th>A-form</th>
<th>B-form</th>
<th>Z-form</th>
</tr>
<tr>
<td>Helix sense</td>
<td align="center">right-handed</td>
<td align="center">right-handed</td>
<td align="center">left-handed</td>
</tr>
<tr bgcolor="#EFEFEF">
<td>Repeating unit</td>
<td align="right">1 bp</td>
<td align="right">1 bp</td>
<td align="right">2 bp</td>
</tr>
<tr>
<td>Rotation/bp</td>
<td align="right">33.6°</td>
<td align="right">35.9° <small>(±4.2°)</small></td>
<td align="right">60°/2</td>
</tr>
<tr bgcolor="#EFEFEF">
<td>Mean bp/turn</td>
<td align="right">10.7</td>
<td align="right">10.0 <small>(±1.2)</small></td>
<td align="right">12</td>
</tr>
<tr>
<td>Inclination of bp to axis</td>
<td align="right">+19°</td>
<td align="right">-1.2° <small>(±4.1°)</small></td>
<td align="right">-9°</td>
</tr>
<tr bgcolor="#EFEFEF">
<td>Rise/bp along axis</td>
<td align="right">0.23 nm</td>
<td align="right">0.332 nm <small>(±0.019nm)</small></td>
<td align="right">0.38 nm</td>
</tr>
<tr>
<td>Pitch/turn of helix</td>
<td align="right">2.46 nm</td>
<td align="right">3.32 nm <small>(±0.19nm)</small></td>
<td align="right">4.56 nm</td>
</tr>
<tr bgcolor="#EFEFEF">
<td>Mean propeller twist</td>
<td align="right">+18°</td>
<td align="right">+16° <small>( ±7°)</small></td>
<td align="right">0°</td>
</tr>
<tr>
<td>Glycosyl angle</td>
<td align="center">anti</td>
<td align="center">anti</td>
<td align="center">C: anti,<br />
G: syn</td>
</tr>
<tr bgcolor="#EFEFEF">
<td>Sugar pucker</td>
<td align="center">C3'-endo</td>
<td align="center">C2'-endo</td>
<td align="center">C: C2'-endo,<br />
G: C2'-exo</td>
</tr>
<tr>
<td>Diameter</td>
<td align="right">2.55 nm</td>
<td align="right">2.37 nm</td>
<td align="right">1.84 nm</td>
</tr>
<tr>
<td colspan="4" style="border-top: 1px solid #333;"><small>Note: Adapted from <i>Table 1</i> in Dickerson et al., <i>Science</i> 1982<sup id="_ref-science1982-Dickerson_0" class="reference"><a  title="">[5]</a></sup>. Values in parentheses represent the standard deviation from three independently prepared <a href="/wiki/X-ray_crystallography.html" title="X-ray crystallography">X-ray crystal</a> samples.</small></td>
</tr>
</table>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Non-helical forms">edit</a>]</div>
<p><a name="Non-helical_forms" id="Non-helical_forms"></a></p>
<h3>Non-helical forms</h3>
<p>There is an argument to be made that the native, intracellular form of DNA is not the B-form double helix, as commonly supposed. Rather, this argument proposes, the strands of DNA remain almost entirely topologically unlinked in their normal states. Watson-Crick base pairing is retained to link the two anti-parallel helical strands together but the two helices do not wind around each other with a net topological linkage when they are in a relaxed topological state.</p>
<p>The double helix, of course, in the B form, has one topological link every 10 base pairs, which is why a credible mode of unwinding the double helix held in a tightly packed three-dimensional ball inside a eukaryotic cell nucleus has remained a topic of continuing interest since the model was proposed in 1953.</p>
<p>Information on this alternative theory is available from this online book, presented in PDF format:</p>
<p><a  class="external free" title="http://www.notahelix.com/delmonte/new_struct_mol_biol.pdf">http://www.notahelix.com/delmonte/new_struct_mol_biol.pdf</a></p>
<p>and a recent research paper summarises some key experimental data which are better explained by SBS models than by the double helix:</p>
<p><a  class="external free" title="http://www.ias.ac.in/currsci/dec102003/1564.pdf">http://www.ias.ac.in/currsci/dec102003/1564.pdf</a></p>
<p>with subsequent correspondence:</p>
<p><a  class="external free" title="http://www.ias.ac.in/currsci/may252004/1352.pdf">http://www.ias.ac.in/currsci/may252004/1352.pdf</a></p>
<p>However, these theories have been seen by some as having problems of their own, such as explaining the near-perfect symmetry of DNA in cells and the activity of DNA repair in the absence of a base-paired strand for comparison.</p>
<p>It is important to note that the Watson-Crick base pairing scheme is retained in side-by-side models of DNA structure, nevertheless, so the high symmetry of complementary base pairs in replication, for example, is retained. The key feature of the paranemic side-by-side model of duplex DNA is that it demonstrates how DNA operator sequences, antibody sensitivities, and target DNA sequences in general, are made rapidly and easily accessible, whereas the double helix model requires great lengths of duplex DNA to be partially unwound before such target sequences can be found since they are largely hidden and buried with the double helix model.</p>
<p>Additionally, the activity of <a href="/wiki/Topoisomerase.html" title="Topoisomerase">topoisomerases</a> would be entirely redundant, and not nearly as important to cellular function as it patently is, if not for the fact that base-paired double-strands are at least the primary form of cellular DNA.</p>
<p>The counterview, based upon the side-by-side models, however, is not that the functions of topoisomerases would be redundant. Even with side-by-side models there is still the phenomenon of supercoiling which is seen to be a valid topological constraint in cellular biochemical processes and enzymes such as the topoisomerases are still needed to separate DNA strands held together by supercoils.</p>
<p>The most demanding phenomenon for the double helix model to explain was reported inside solid fibers as early as 1973. Leslie et al. (1) reported that a fibre of poly(dI).poly(dC), in the absence of unwinding proteins, which gave sharp X-ray diffraction spots shortly after being drawn, was unstable and transformed irreversibly after a few days into poly(dC).poly(dI).poly(dC+). This new, three-stranded molecule also gave sharp spots in its own, different X-ray diffraction pattern which did not have any of the original spots.</p>
<p>Therefore the conversion was complete or substantially so and the product was both highly ordered and crystalline. Very similar results had been reported for fibres of poly(dA).poly(dT) converting to poly(dT).poly(dA).poly(dT) (2) and for the formation of poly(U).poly(A).poly(U) from poly(U).poly(A) (3).</p>
<p>The problem is evident: How and where would the torque, a vector, arise, and be sustained, inside a solid fibre that would permit one double helix to rotate against the friction of its neighbours so as to unwind one strand of an adjacent duplex and rewind it onto a triple helix having a different diameter and rotating at a different angular velocity, the conversion clearly being complete or largely so&#160;?</p>
<p>Inside the fibre, individual molecules would have a random axial translation so only very rarely would two adjacent double helices have the same axial starting point down the fibre. They would be unlikely to be perfectly straight inside the fibre but would probably be at least partly entwined around neighbours, or at least twisted away from a true linear disposition. Moreover, any particular duplex would be equidistant from several neighbours, and, along its length, it could start to form a triplex by unwinding different neighbours.</p>
<p>A true, paranemic, antiparallel, side-by-side duplex DNA structure could form a side-by-side triplex, without any unwinding, by the linear transfer of a strand across to a neighbouring duplex.</p>
<p>1 Leslie, A. G. W., Arnott, S., Chandrasekaran, R. &amp; Ratliff, R. L., J. Mol. Biol. 143, 49 - 72 (1980)</p>
<p>2 Arnott, S. &amp; E. Selsing, E., J. Mol. Biol. 88, 509-521 (1974)</p>
<p>3 Arnott, S. &amp; Bond, P. J., Nature New Biology 244, 99 -101 (1973)</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Strand direction">edit</a>]</div>
<p><a name="Strand_direction" id="Strand_direction"></a></p>
<h2>Strand direction</h2>
<p>The asymmetric shape and linkage of nucleotides means that a DNA strand always has a discernible orientation or directionality. Because of this directionality, close inspection of a double helix reveals that nucleotides are heading one way along one strand (the "<i>ascending strand</i>"), and the other way along the other strand (the "<i>descending strand</i>"). This arrangement of the strands is called <b>antiparallel</b>.</p>
<dl>
<dt>Chemical nomenclature (<a href="/wiki/5%27_end.html" title="5' end">5'</a> and <a href="/wiki/3%27_end.html" title="3' end">3'</a>)</dt>
</dl>
<p>For reasons of chemical nomenclature, people who work with DNA refer to the asymmetric ends of ("five prime" and "three prime"). Within a cell, the enzymes that perform <a href="/wiki/DNA_replication.html" title="DNA replication">replication</a> and <a href="/wiki/DNA_transcription.html" title="DNA transcription">transcription</a> read DNA in the "<b><a href="/wiki/3%27_end.html" title="3' end">3'</a> to <a href="/wiki/5%27_end.html" title="5' end">5'</a> direction</b>", while the enzymes that perform translation read in the opposite directions (on <a href="/wiki/RNA.html" title="RNA">RNA</a>). However, because chemically produced DNA is synthesized and manipulated in the opposite or in non-directional manners, the orientation should not be assumed. In a vertically oriented double helix, the <a href="/wiki/3%27_end.html" title="3' end">3'</a> strand is said to be ascending while the <a href="/wiki/5%27_end.html" title="5' end">5'</a> strand is said to be descending.</p>
<dl>
<dt>Sense and antisense</dt>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/Sense_%28molecular_biology%29.html" title="Sense (molecular biology)">Sense (molecular biology)</a></i></div>
</dd>
</dl>
<p>As a result of their antiparallel arrangement and the sequence-reading preferences of enzymes, even if both strands carried identical instead of complementary sequences, cells could properly translate only one of them. The other strand a cell can only read backwards. <a href="/wiki/Molecular_biology.html" title="Molecular biology">Molecular biologists</a> call a sequence "<b>sense</b>" if it is translated or translatable, and they call its complement "<b>antisense</b>". It follows then, somewhat paradoxically, that the template for transcription is the <i>antisense</i> strand. The resulting transcript is an RNA replica of the <i>sense</i> strand and is itself <i>sense.</i></p>
<p>A small proportion of genes in <a href="/wiki/Prokaryotes.html" title="Prokaryotes">prokaryotes</a>, and more in <a href="/wiki/Plasmids.html" title="Plasmids">plasmids</a> and <a href="/wiki/Viruses.html" title="Viruses">viruses</a>, blur the distinction made above between sense and antisense strands. Certain sequences of their <a href="/wiki/Genome.html" title="Genome">genomes</a> do double duty, encoding one protein when read 5' to 3' along one strand, and a second protein when read in the opposite direction (still 5' to 3') along the other strand. As a result, the genomes of these viruses are unusually compact for the number of genes they contain, which biologists view as an <a href="/wiki/Adaptation_%28biology%29.html" title="Adaptation (biology)">adaptation</a>. This merely confirms that there is no biological distinction between the two strands of the double helix. Typically each strand of a DNA double helix will act as sense and antisense in different regions.</p>
<dl>
<dt>As viewed by topologists</dt>
</dl>
<p>Topologists like to note that the juxtaposition of the <a href="/wiki/3%E2%80%B2_end.html" title="3′ end">3′ end</a> of one DNA strand beside the <a href="/wiki/5%E2%80%B2_end.html" title="5′ end">5′ end</a> of the other at both ends of a double-helical segment makes the arrangement a "<a href="/wiki/Crab_canon.html" title="Crab canon">crab canon</a>".</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Single-stranded DNA (ssDNA) and repair of mutations">edit</a>]</div>
<p><a name="Single-stranded_DNA_.28ssDNA.29_and_repair_of_mutations" id="Single-stranded_DNA_.28ssDNA.29_and_repair_of_mutations"></a></p>
<h2>Single-stranded DNA (ssDNA) and repair of mutations</h2>
<p>In some <a href="/wiki/Virus.html" title="Virus">viruses</a> DNA appears in a non-helical, single-stranded form. Because many of the <a href="/wiki/DNA_repair.html" title="DNA repair">DNA repair</a> mechanisms of cells work only on paired bases, viruses that carry single-stranded DNA <a href="/wiki/Genome.html" title="Genome">genomes</a> <a href="/wiki/Mutation.html" title="Mutation">mutate</a> more frequently than they would otherwise. As a result, such species may adapt more rapidly to avoid extinction. The result would not be so favorable in more complicated and more slowly replicating organisms, however, which may explain why only viruses carry single-stranded DNA. These viruses presumably also benefit from the lower cost of replicating one strand versus two.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: History of DNA research">edit</a>]</div>
<p><a name="History_of_DNA_research" id="History_of_DNA_research"></a></p>
<h2>History of DNA research</h2>
<div class="thumb tright">
<div style="width:202px;"><a  class="internal" title="James Watson in the Cavendish Laboratory at the University of Cambridge"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/2/2b/JamesWatson.jpg/200px-JamesWatson.jpg" alt="James Watson in the Cavendish Laboratory at the University of Cambridge" width="200" height="249" longdesc="/wiki/Image:JamesWatson.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
<a href="/wiki/James_D._Watson.html" title="James D. Watson">James Watson</a> in the <a href="/wiki/Cavendish_Laboratory.html" title="Cavendish Laboratory">Cavendish Laboratory</a> at the <a href="/wiki/University_of_Cambridge.html" title="University of Cambridge">University of Cambridge</a></div>
</div>
</div>
<p>The discovery that DNA was the carrier of genetic information was a process that required many earlier discoveries. The existence of DNA was discovered in the mid 19th century. However, it was only in the early 20th century that researchers began suggesting that it might store genetic information. This gained almost universal acceptance after the structure of DNA was elucidated by <a href="/wiki/James_D._Watson.html" title="James D. Watson">James D. Watson</a> and <a href="/wiki/Francis_Crick.html" title="Francis Crick">Francis Crick</a> in their 1953 <a href="/wiki/Nature_%28journal%29.html" title="Nature (journal)"><i>Nature</i></a> publication. Watson and Crick proposed the <a href="/wiki/Central_dogma.html" title="Central dogma">central dogma</a> of molecular biology in 1957, describing the process whereby proteins are produced from <a href="/wiki/Cell_nucleus.html" title="Cell nucleus">nucleic</a> DNA. In 1962 Watson, Crick, and <a href="/wiki/Maurice_Wilkins.html" title="Maurice Wilkins">Maurice Wilkins</a> jointly received the <a href="/wiki/Nobel_Prize.html" title="Nobel Prize">Nobel Prize</a> for their determination of the structure of DNA.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: First isolation of DNA">edit</a>]</div>
<p><a name="First_isolation_of_DNA" id="First_isolation_of_DNA"></a></p>
<h3>First isolation of DNA</h3>
<p>Working in the 19th century, biochemists initially isolated DNA and RNA (mixed together) from cell nuclei. They were relatively quick to appreciate the polymeric nature of their "nucleic acid" isolates, but realized only later that nucleotides were of two types--one containing <a href="/wiki/Ribose.html" title="Ribose">ribose</a> and the other <a href="/wiki/Deoxyribose.html" title="Deoxyribose">deoxyribose</a>. It was this subsequent discovery that led to the identification and naming of DNA as a substance distinct from RNA.</p>
<p><a href="/wiki/Friedrich_Miescher.html" title="Friedrich Miescher">Friedrich Miescher</a> (1844-1895) discovered a substance he called "nuclein" in 1869. Somewhat later, he isolated a pure sample of the material now known as DNA from the sperm of salmon, and in 1889 his pupil, <a href="/wiki/Richard_Altmann.html" title="Richard Altmann">Richard Altmann</a>, named it "nucleic acid". This substance was found to exist only in the chromosomes.</p>
<p>In 1929 <a href="/wiki/Phoebus_Levene.html" title="Phoebus Levene">Phoebus Levene</a> at the <a href="/wiki/Rockefeller_Institute.html" title="Rockefeller Institute">Rockefeller Institute</a> identified the components (the four bases, the sugar and the phosphate chain) and he showed that the components of DNA were linked in the order phosphate-sugar-base. He called each of these units a <a href="/wiki/Nucleotide.html" title="Nucleotide">nucleotide</a> and suggested the DNA molecule consisted of a string of nucleotide units linked together through the phosphate groups, which are the 'backbone' of the molecule. However Levene thought the chain was short and that the bases repeated in the same fixed order. <a href="/wiki/Torbjorn_Oskar_Caspersson.html" title="Torbjorn Oskar Caspersson">Torbjorn Caspersson</a> and <a  class="new" title="Einar Hammersten">Einar Hammersten</a> showed that DNA was a polymer.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Chromosomes and inherited traits">edit</a>]</div>
<p><a name="Chromosomes_and_inherited_traits" id="Chromosomes_and_inherited_traits"></a></p>
<h3>Chromosomes and inherited traits</h3>
<p><a href="/wiki/Max_Delbr%C3%BCck.html" title="Max Delbrück">Max Delbrück</a>, <a href="/wiki/Nikolai_V._Timofeeff-Ressovsky.html" title="Nikolai V. Timofeeff-Ressovsky">Nikolai V. Timofeeff-Ressovsky</a>, and <a href="/wiki/Karl_G._Zimmer.html" title="Karl G. Zimmer">Karl G. Zimmer</a> published results in 1935 suggesting that chromosomes are very large molecules the structure of which can be changed by treatment with <a href="/wiki/X-ray.html" title="X-ray">X-rays</a>, and that by so changing their structure it was possible to change the heritable characteristics governed by those chromosomes. In 1937 <a href="/wiki/William_Astbury.html" title="William Astbury">William Astbury</a> produced the first <a href="/wiki/X-ray_diffraction.html" title="X-ray diffraction">X-ray diffraction</a> patterns from DNA. He was not able to propose the correct structure but the patterns showed that DNA had a regular structure and therefore it might be possible to deduce what this structure was.</p>
<p>In 1943, <a href="/wiki/Oswald_Theodore_Avery.html" title="Oswald Theodore Avery">Oswald Theodore Avery</a> and a team of scientists discovered that traits proper to the "smooth" form of the <i>Pneumococcus</i> could be transferred to the "rough" form of the same bacteria merely by making the killed "smooth" (S) form available to the live "rough" (R) form. Quite unexpectedly, the living R <i>Pneumococcus</i> bacteria were transformed into a new strain of the S form, and the transferred S characteristics turned out to be heritable. Avery called the medium of transfer of traits the <a href="/wiki/Transforming_principle.html" title="Transforming principle">transforming principle</a>; he identified DNA as the transforming principle, and not <a href="/wiki/Protein.html" title="Protein">protein</a> as previously thought. He essentially redid <a  class="new" title="Fredrick Griffith">Fredrick Griffith</a>'s experiment. In 1953, <a href="/wiki/Alfred_Hershey.html" title="Alfred Hershey">Alfred Hershey</a> and <a href="/wiki/Martha_Chase.html" title="Martha Chase">Martha Chase</a> did an experiment (<a href="/wiki/Hershey-Chase_experiment.html" title="Hershey-Chase experiment">Hershey-Chase experiment</a>) that showed, in <a href="/wiki/T2_phage.html" title="T2 phage">T2 phage</a>, that DNA is the <a href="/wiki/Genetic_material.html" title="Genetic material">genetic material</a> (Hershey shared the Nobel prize with Luria).</p>
<div class="thumb tright">
<div style="width:202px;"><a  class="internal" title="Francis Crick's first sketch of the deoxyribonucleic acid double-helix pattern"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/c/c8/FirstSketchOfDNADoubleHelix.jpg/200px-FirstSketchOfDNADoubleHelix.jpg" alt="Francis Crick's first sketch of the deoxyribonucleic acid double-helix pattern" width="200" height="333" longdesc="/wiki/Image:FirstSketchOfDNADoubleHelix.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
<a href="/wiki/Francis_Crick.html" title="Francis Crick">Francis Crick</a>'s first sketch of the <a href="/wiki/Deoxyribonucleic_acid.html" title="Deoxyribonucleic acid">deoxyribonucleic acid</a> double-helix pattern</div>
</div>
</div>
<p>In 1944, the renowned physicist, <a href="/wiki/Erwin_Schr%C3%B6dinger.html" title="Erwin Schrödinger">Erwin Schrödinger</a>, published a brief book entitled <i><a href="/wiki/What_is_Life%3F_%28Schr%C3%B6dinger%29.html" title="What is Life? (Schrödinger)">What is Life?</a></i>, where he maintained that chromosomes contained what he called the "hereditary code-script" of life. He added: "But the term code-script is, of course, too narrow. The chromosome structures are at the same time instrumental in bringing about the development they foreshadow. They are law-code and executive power -- or, to use another simile, they are architect's plan and builder's craft -- in one." He conceived of these dual functional elements as being woven into the molecular structure of chromosomes. By understanding the exact molecular structure of the chromosomes one could hope to understand both the "architect's plan" and also how that plan was carried out through the "builder's craft." Three groups took up Schrödinger's challenge to work out the structure of the chromosomes and the question of how the segments of the chromosomes that were conceived to relate to specific traits could possibly do their jobs.</p>
<p>Just how the presence of specific features in the molecular structure of chromosomes could produce traits and behaviors in living organisms was unimaginable at the time. Because chemical dissection of DNA samples always yielded the same four nucleotides, the chemical composition of DNA appeared simple, perhaps even uniform. Organisms, on the other hand, are fantastically complex individually and widely diverse collectively. Geneticists did not speak of genes as conveyors of "information" in such words, but if they had, they would not have hesitated to quantify the amount of information that genes need to convey as vast. The idea that information might reside in a chemical in the same way that it exists in text--as a finite alphabet of letters arranged in a sequence of unlimited length--had not yet been conceived. It would emerge upon the discovery of DNA's structure, but few researchers imagined that DNA's structure had much to say about genetics.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Discovery of the structure of DNA">edit</a>]</div>
<p><a name="Discovery_of_the_structure_of_DNA" id="Discovery_of_the_structure_of_DNA"></a></p>
<h3>Discovery of the structure of DNA</h3>
<p>In the 1950s, three groups made it their goal to determine the structure of DNA. The first group to start was at <a href="/wiki/King%27s_College_London.html" title="King's College London">King's College London</a> and was led by <a href="/wiki/Maurice_Wilkins.html" title="Maurice Wilkins">Maurice Wilkins</a> and was later joined by <a href="/wiki/Rosalind_Franklin.html" title="Rosalind Franklin">Rosalind Franklin</a>. Another group consisting of <a href="/wiki/Francis_Crick.html" title="Francis Crick">Francis Crick</a> and <a href="/wiki/James_D._Watson.html" title="James D. Watson">James D. Watson</a> was at <a href="/wiki/University_of_Cambridge.html" title="University of Cambridge">Cambridge</a>. A third group was at <a href="/wiki/Caltech.html" title="Caltech">Caltech</a> and was led by <a href="/wiki/Linus_Pauling.html" title="Linus Pauling">Linus Pauling</a>. Crick and Watson built physical models using metal rods and balls, in which they incorporated the known chemical structures of the nucleotides, as well as the known position of the linkages joining one nucleotide to the next along the polymer. At King's College Maurice Wilkins and Rosalind Franklin examined <a href="/wiki/Crystallography.html" title="Crystallography">X-ray diffraction</a> patterns of DNA fibers. Of the three groups, only the London group was able to produce good quality diffraction patterns and thus produce sufficient quantitative data about the structure.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Helix structure">edit</a>]</div>
<p><a name="Helix_structure" id="Helix_structure"></a></p>
<h4>Helix structure</h4>
<p>In 1948 Pauling discovered that many proteins included helical (see <a href="/wiki/Alpha_helix.html" title="Alpha helix">alpha helix</a>) shapes. Pauling had deduced this structure from X-ray patterns and from attempts to physically model the structures. (Pauling was also later to suggest an incorrect three chain helical structure based on Astbury's data.) Even in the initial diffraction data from DNA by Maurice Wilkins, it was evident that the structure involved helices. But this insight was only a beginning. There remained the questions of how many strands came together, whether this number was the same for every helix, whether the bases pointed toward the helical axis or away, and ultimately what were the explicit angles and coordinates of all the bonds and atoms. Such questions motivated the modeling efforts of Watson and Crick.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Complementary nucleotides">edit</a>]</div>
<p><a name="Complementary_nucleotides" id="Complementary_nucleotides"></a></p>
<h4>Complementary nucleotides</h4>
<p>In their modeling, Watson and Crick restricted themselves to what they saw as chemically and biologically reasonable. Still, the breadth of possibilities was very wide. A breakthrough occurred in 1952, when <a href="/wiki/Erwin_Chargaff.html" title="Erwin Chargaff">Erwin Chargaff</a> visited Cambridge and inspired Crick with a description of experiments Chargaff had published in 1947. Chargaff had observed that the proportions of the four nucleotides vary between one DNA sample and the next, but that for particular pairs of nucleotides — adenine and thymine, guanine and cytosine — the two nucleotides are always present in equal proportions.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Watson and Crick's model">edit</a>]</div>
<p><a name="Watson_and_Crick.27s_model" id="Watson_and_Crick.27s_model"></a></p>
<h4>Watson and Crick's model</h4>
<div class="thumb tright">
<div style="width:202px;"><a  class="internal" title="Crick and Watson DNA model built in 1953, was reconstructed largely from its original pieces in 1973 and donated to the National Science Museum in London."><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/5/50/DNA_Model_Crick-Watson.jpg/200px-DNA_Model_Crick-Watson.jpg" alt="Crick and Watson DNA model built in 1953, was reconstructed largely from its original pieces in 1973 and donated to the National Science Museum in London." width="200" height="267" longdesc="/wiki/Image:DNA_Model_Crick-Watson.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Crick and Watson DNA model built in 1953, was <a  class="external text" title="http://www.farooqhussain.org/projects/">reconstructed</a> largely from its original pieces in 1973 and donated to the <a href="/wiki/Science_Museum_%28London%29.html" title="Science Museum (London)">National Science Museum</a> in London.</div>
</div>
</div>
<p><a href="/wiki/James_D._Watson.html" title="James D. Watson">Watson</a> and <a href="/wiki/Francis_Crick.html" title="Francis Crick">Crick</a> had begun to contemplate double helical arrangements, but they lacked information about the amount of twist (pitch) and the distance between the two strands. <a href="/wiki/Rosalind_Franklin.html" title="Rosalind Franklin">Rosalind Franklin</a> had to disclose some of her findings for the <a href="/wiki/Medical_Research_Council.html" title="Medical Research Council">Medical Research Council</a> and Crick saw this material through <a href="/wiki/Max_Perutz.html" title="Max Perutz">Max Perutz's</a> links to the MRC. Franklin's work confirmed that the phosphate "backbone" was on the outside of the molecule and also gave an insight into its symmetry, in particular that the two helical strands ran in opposite directions.</p>
<p>Watson and Crick were again greatly assisted by more of Franklin's data. This is controversial because Franklin's critical X-ray pattern was shown to Watson and Crick without Franklin's knowledge or permission. Wilkins showed the famous Photo 51 of the much simpler <i>B</i> type of DNA to Watson at his lab immediately after Watson had been unsuccessful in asking Franklin to collaborate to beat Pauling in finding the structure.</p>
<p>From the data in photograph 51 Watson and Crick were able to discern that not only was the distance between the two strands constant, but also to measure its exact value of 2 nanometres. The same photograph also gave them the 3.4 nanometre-per-10 bp "pitch" of the helix.</p>
<p>The final insight came when Crick and Watson saw that a complementary pairing of the bases could provide an explanation for Chargaff's puzzling finding. However the structure of the bases had been incorrectly guessed in the textbooks as the <a href="/wiki/Enol.html" title="Enol">enol</a> <a href="/wiki/Tautomer.html" title="Tautomer">tautomer</a> when they were more likely to be in the <a href="/wiki/Keto.html" title="Keto">keto</a> form. When <a href="/wiki/Jerry_Donohue.html" title="Jerry Donohue">Jerry Donohue</a> pointed this fallacy out to Watson, Watson quickly realised that the pairs of adenine and thymine, and guanine and cytosine were almost identical in shape and so would provide equally sized 'rungs' between the two strands. Watson and Crick worked to develop a physical model of the double-helical structure out of wire which they used to confirm that the distances between the molecules were permissible. With the base-pairing, the Watson and Crick quickly converged upon a model, which they announced before Franklin herself had published any of her work.</p>
<p>Franklin was herself two steps away from the solution. She had not guessed the base-pairing and had not appreciated the implications of the symmetry that she had described. However she had been working almost alone and did not have regular contact with a partner like Crick and Watson, and with other experts such as Jerry Donohue. Her notebooks show that she was aware both of Jerry Donohue's work concerning tautomeric forms of bases (she had used the keto forms for three of the bases) and of Chargaff's work.</p>
<p>The disclosure of Franklin's data to Watson has angered some people who believe Franklin did not receive due credit at the time and that she might have discovered the structure on her own before Crick and Watson. In Crick and Watson's famous paper in Nature in 1953, they said that their work had been stimulated by the work of Wilkins and Franklin, whereas it had been the basis of their work. However they had agreed with Wilkins and Franklin that they all should publish papers in the same issue of <i>Nature</i> in support of the proposed structure. Additionally, in his autobiography, <i><a href="/wiki/The_Double_Helix.html" title="The Double Helix">The Double Helix</a></i>, Watson describes Franklin in very unflattering terms (commenting derisively on her lack of "feminine" traits) and all but implies that her work actually impaired that of Wilkins.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: &quot;Central Dogma&quot;">edit</a>]</div>
<p><a name=".22Central_Dogma.22"></a></p>
<h4>"Central Dogma"</h4>
<p>Watson and Crick's model attracted great interest immediately upon its presentation. Arriving at their conclusion on <a href="/wiki/February_21.html" title="February 21">February 21</a>, <a href="/wiki/1953.html" title="1953">1953</a>, Watson and Crick made their first announcement on <a href="/wiki/February_28.html" title="February 28">February 28</a>. Their paper, <i>A Structure for Deoxyribose Nucleic Acid</i>,<sup id="_ref-nature1953-watson_0" class="reference"><a  title="">[6]</a></sup> was published on April 25. In an influential presentation in 1957, Crick laid out the "<a href="/wiki/Central_Dogma.html" title="Central Dogma">Central Dogma</a>", which foretold the relationship between DNA, RNA, and proteins, and articulated the "sequence hypothesis." A critical confirmation of the replication mechanism that was implied by the double-helical structure followed in 1958 in the form of the <a href="/wiki/Meselson-Stahl_experiment.html" title="Meselson-Stahl experiment">Meselson-Stahl experiment</a>. Work by Crick and coworkers showed that the genetic code was based on non-overlapping triplets of bases, called codons, and <a href="/wiki/Har_Gobind_Khorana.html" title="Har Gobind Khorana">Har Gobind Khorana</a> and others deciphered the <a href="/wiki/Genetic_code.html" title="Genetic code">genetic code</a> not long afterward. These findings represent the birth of <a href="/wiki/Molecular_biology.html" title="Molecular biology">molecular biology</a>.</p>
<p><a href="/wiki/James_D._Watson.html" title="James D. Watson">Watson</a>, <a href="/wiki/Francis_Crick.html" title="Francis Crick">Crick</a>, and <a href="/wiki/Maurice_Wilkins.html" title="Maurice Wilkins">Wilkins</a> were awarded the 1962 <a href="/wiki/Nobel_Prize_for_Physiology_or_Medicine.html" title="Nobel Prize for Physiology or Medicine">Nobel Prize for Physiology or Medicine</a> for discovering the molecular structure of DNA, by which time <a href="/wiki/Rosalind_Franklin.html" title="Rosalind Franklin">Franklin</a> had died from cancer at 37. Nobel prizes are not awarded posthumously; had she lived, the difficult decision over whom to jointly award the prize would have been complicated as the prize can only be shared between a maximum of three; but because their work could be considered to be chemistry, it is conceivable that <a href="/wiki/Maurice_Wilkins.html" title="Maurice Wilkins">Wilkins</a> and <a href="/wiki/Rosalind_Franklin.html" title="Rosalind Franklin">Franklin</a> could have been awarded the <a href="/wiki/Nobel_Prize_for_Chemistry.html" title="Nobel Prize for Chemistry">Nobel Prize for Chemistry</a> instead; see Graeme Hunter's biography of Sir Lawrence Bragg for more information on how scientists were nominated for Nobel Prizes.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: DNA in practice">edit</a>]</div>
<p><a name="DNA_in_practice" id="DNA_in_practice"></a></p>
<h2>DNA in practice</h2>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: DNA in crime">edit</a>]</div>
<p><a name="DNA_in_crime" id="DNA_in_crime"></a></p>
<h3>DNA in crime</h3>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/Genetic_fingerprinting.html" title="Genetic fingerprinting">Genetic fingerprinting</a></i></div>
</dd>
</dl>
<p><a href="/wiki/Forensic_science.html" title="Forensic science">Forensic scientists</a> can use DNA located in <a href="/wiki/Blood.html" title="Blood">blood</a>, <a href="/wiki/Semen.html" title="Semen">semen</a>, <a href="/wiki/Skin.html" title="Skin">skin</a>, <a href="/wiki/Saliva.html" title="Saliva">saliva</a> or hair left at the scene of a crime to identify a possible suspect, a process called <a href="/wiki/Genetic_fingerprinting.html" title="Genetic fingerprinting">genetic fingerprinting</a> or DNA profiling. In DNA profiling the relative lengths of sections of repetitive DNA, such as <a href="/wiki/Short_tandem_repeats.html" title="Short tandem repeats">short tandem repeats</a> and <a href="/wiki/Minisatellite.html" title="Minisatellite">minisatellites</a>, are compared. DNA profiling was developed in 1984 by English geneticist <a  class="new" title="Sir Alec Jeffreys">Sir Alec Jeffreys</a> of the <a href="/wiki/University_of_Leicester.html" title="University of Leicester">University of Leicester</a>, and was first used to convict Colin Pitchfork in 1988 in the <a href="/wiki/Enderby_murders.html" title="Enderby murders">Enderby murders</a> case in <a href="/wiki/Leicestershire.html" title="Leicestershire">Leicestershire</a>, <a href="/wiki/England.html" title="England">England</a>. Many jurisdictions require convicts of certain types of crimes to provide a sample of DNA for inclusion in a computerized database. This has helped investigators solve old cases where the perpetrator was unknown and only a DNA sample was obtained from the scene (particularly in <a href="/wiki/Rape.html" title="Rape">rape</a> cases between strangers). This method is one of the most reliable techniques for identifying a criminal, but is not always perfect, for example if no DNA can be retrieved, or if the scene is contaminated with the DNA of several possible suspects.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: DNA in computation">edit</a>]</div>
<p><a name="DNA_in_computation" id="DNA_in_computation"></a></p>
<h3>DNA in computation</h3>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/DNA_computing.html" title="DNA computing">DNA computing</a></i></div>
</dd>
</dl>
<p>DNA plays an important role in <a href="/wiki/Computer_science.html" title="Computer science">computer science</a>, <a href="/wiki/Bioinformatics.html" title="Bioinformatics">bioinformatics and computational biology</a>, both as a motivating research problem and as a method of computation in itself.</p>
<p>Research on <a href="/wiki/String_searching_algorithm.html" title="String searching algorithm">string searching algorithms</a>, which find an occurrence of a sequence of letters inside a larger sequence of letters, was motivated in part by DNA research, where it is used to find specific sequences of nucleotides in a large sequence.<sup id="_ref-2" class="reference"><a  title="">[7]</a></sup> In other applications such as <a href="/wiki/Text_editor.html" title="Text editor">text editors</a>, even simple algorithms for this problem usually suffice, but DNA sequences cause these algorithms to exhibit near-worst-case behavior due to their small number of distinct characters.</p>
<p><a href="/wiki/Database.html" title="Database">Database</a> theory has been influenced by DNA research, which poses special problems for storing and manipulating DNA sequences. Databases specialized for DNA research are called <a  class="new" title="Genomic database">genomic databases</a>, and must address a number of unique technical challenges associated with the operations of approximate matching, sequence comparison, finding repeating patterns, and homology searching.</p>
<p>In 1994, <a href="/wiki/Leonard_Adleman.html" title="Leonard Adleman">Leonard Adleman</a> of the <a href="/wiki/University_of_Southern_California.html" title="University of Southern California">University of Southern California</a> made headlines when he discovered a way of solving the directed <a href="/wiki/Hamiltonian_path_problem.html" title="Hamiltonian path problem">Hamiltonian path problem</a>, an <a href="/wiki/NP-complete.html" title="NP-complete">NP-complete</a> problem, using tools from molecular biology, in particular DNA. The new approach, dubbed <a href="/wiki/DNA_computing.html" title="DNA computing">DNA computing</a>, has practical advantages over traditional computers in power use, space use, and efficiency, due to its ability to highly parallelize the computation (see <a href="/wiki/Parallel_computing.html" title="Parallel computing">parallel computing</a>), although there is labor worth mentioning involved in retrieving the answers. A number of other problems, including simulation of various <a href="/wiki/Abstract_machine.html" title="Abstract machine">abstract machines</a>, the <a href="/wiki/Boolean_satisfiability_problem.html" title="Boolean satisfiability problem">boolean satisfiability problem</a>, and the bounded version of the <a href="/wiki/Post_correspondence_problem.html" title="Post correspondence problem">Post correspondence problem</a>, have since been analyzed using DNA computing.</p>
<p>Due to its compactness, DNA also has a theoretical role in <a href="/wiki/Cryptography.html" title="Cryptography">cryptography</a>, where in particular it allows unbreakable <a href="/wiki/One-time_pad.html" title="One-time pad">one-time pads</a> to be efficiently constructed and used.<sup id="_ref-3" class="reference"><a  title="">[8]</a></sup></p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: DNA in historical and anthropological study">edit</a>]</div>
<p><a name="DNA_in_historical_and_anthropological_study" id="DNA_in_historical_and_anthropological_study"></a></p>
<h3>DNA in historical and anthropological study</h3>
<p>Because DNA collects mutations over time, which are then passed down from parent to offspring, it contains information about processes that have occurred in the past, becoming in time <a href="/wiki/Ancient_DNA.html" title="Ancient DNA">ancient DNA</a>. By comparing different DNA sequences, geneticists can attempt to infer the history of organisms.</p>
<p>If DNA sequences from different <a href="/wiki/Species.html" title="Species">species</a> are compared, then the resulting family tree, or <a href="/wiki/Phylogeny.html" title="Phylogeny">phylogeny</a> can be used to study the <a href="/wiki/Evolution.html" title="Evolution">evolution</a> of these species. This field of <a href="/wiki/Phylogenetics.html" title="Phylogenetics">phylogenetics</a> is a powerful tool in <a href="/wiki/Evolutionary_biology.html" title="Evolutionary biology">evolutionary biology</a>. If DNA sequences within a species are compared, <a href="/wiki/Population_genetics.html" title="Population genetics">population geneticists</a> can glean information on the history of particular populations. This can be used in studies ranging from <a href="/wiki/Ecological_genetics.html" title="Ecological genetics">ecological genetics</a> to <a href="/wiki/Anthropology.html" title="Anthropology">anthropology</a> (for example, DNA evidence is also being used to try to identify the <a href="/wiki/Ten_Lost_Tribes_of_Israel.html" title="Ten Lost Tribes of Israel">Ten Lost Tribes of Israel</a><sup id="_ref-4" class="reference"><a  title="">[9]</a></sup><sup id="_ref-5" class="reference"><a  title="">[10]</a></sup>).</p>
<p>DNA has also been used to look at fairly recent issues of family relationships, such as establishing some manner of familial relationship between the descendants of <a href="/wiki/Sally_Hemings.html" title="Sally Hemings">Sally Hemings</a> and the family of <a href="/wiki/Thomas_Jefferson.html" title="Thomas Jefferson">Thomas Jefferson</a>. This usage is closely related to the use of DNA in criminal investigations detailed above. Indeed, some criminal investigations have been solved when DNA from crime scenes has fortuitously matched relatives of the guilty individual.<a  class="external autonumber" title="http://www.newscientist.com/article.ns?id=dn4908">[1]</a><a  class="external autonumber" title="http://news.bbc.co.uk/1/hi/wales/3044282.stm">[2]</a></p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: References">edit</a>]</div>
<p><a name="References" id="References"></a></p>
<h2>References</h2>
<dl>
<dt>Citations</dt>
</dl>
<div class="references-small">
<ol class="references">
<li id="_note-0"><b><a  title="">^</a></b> <a  class="external free" title="http://proxy.arts.uci.edu/~nideffer/Hawking/early_proto/orgel.html">http://proxy.arts.uci.edu/~nideffer/Hawking/early_proto/orgel.html</a></li>
<li id="_note-Butler">^ <a  title=""><sup><i><b>a</b></i></sup></a>&#160;<a  title=""><sup><i><b>b</b></i></sup></a>&#160;<a  title=""><sup><i><b>c</b></i></sup></a> Butler, John M. (2001) <i>Forensic DNA Typing</i> "Elsevier". pp. 14-15. <a  class="internal">ISBN 012147951X</a>.</li>
<li id="_note-nature1963-takahashi"><b><a  title="">^</a></b> <cite style="font-style:normal">Takahashi I, Marmur J. (1963). "Replacement of thymidylic acid by deoxyuridylic acid in the deoxyribonucleic acid of a transducing phage for Bacillus subtilis". <i>Nature</i> <b>197</b>: 794-5. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=13980287">PMID 13980287</a>.</cite></li>
<li id="_note-1"><b><a  title="">^</a></b> <a  class="external text" title="http://hypertextbook.com/facts/1998/StevenChen.shtml">Length of a Human DNA Molecule</a>. Retrieved on <a href="/wiki/2006.html" title="2006">2006</a>-<a href="/wiki/March_4.html" title="March 4">03-04</a>.</li>
<li id="_note-science1982-Dickerson"><b><a  title="">^</a></b> <cite style="font-style:normal">Dickerson et al. (1982). "The Anatomy of A-, B-, and Z-DNA". <i>Science</i> <b>216</b>: 475-485. DOI: 10.1126/science.7071593.</cite><sup><a  class="external text" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=7071593&amp;dopt=Abstract">PubMed</a></sup></li>
<li id="_note-nature1953-watson"><b><a  title="">^</a></b> <cite style="font-style:normal">Watson JD, Crick FH. (1953). "Molecular structure of nucleic acids; a structure for deoxyribose nucleic acid". <i>Nature</i> <b>171</b> (4356): 737-8. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=13054692">PMID 13054692</a>.</cite></li>
<li id="_note-2"><b><a  title="">^</a></b> Gusfield, Dan. <i>Algorithms on Strings, Trees, and Sequences: Computer Science and Computational Biology</i>. Cambridge University Press, 15 January <a href="/wiki/1997.html" title="1997">1997</a>. <a  class="internal">ISBN 0521585198</a>.</li>
<li id="_note-3"><b><a  title="">^</a></b> Ashish Gehani, Thomas LaBean and John Reif. <a  class="external text" title="http://citeseer.ist.psu.edu/gehani99dnabased.html">DNA-Based Cryptography</a>. Proceedings of the 5th DIMACS Workshop on DNA Based Computers, Cambridge, MA, USA, 14 – 15 June 1999.</li>
<li id="_note-4"><b><a  title="">^</a></b> <i>Lost Tribes of Israel</i>, <a href="/wiki/NOVA_%28TV_series%29.html" title="NOVA (TV series)">NOVA</a>, PBS airdate: 22 February 2000. Transcript available from <a  class="external free" title="http://www.pbs.org/wgbh/nova/transcripts/2706israel.html">http://www.pbs.org/wgbh/nova/transcripts/2706israel.html</a> (last accessed on 4 March 2006)</li>
<li id="_note-5"><b><a  title="">^</a></b> Kleiman, Yaakov. <a  class="external text" title="http://www.aish.com/societywork/sciencenature/the_cohanim_-_dna_connection.asp">The Cohanim/DNA Connection</a>. Retrieved on <a href="/wiki/2006.html" title="2006">2006</a>-<a href="/wiki/March_4.html" title="March 4">03-04</a>.</li>
</ol>
</div>
<dl>
<dt>General references</dt>
</dl>
<ul>
<li><a href="/wiki/Robert_Olby.html" title="Robert Olby">Robert Olby</a>; "The Path to The Double Helix: Discovery of DNA"; first published in 0ctober 1974 by MacMillan, with foreword by Francis Crick; <a  class="internal">ISBN 046681173</a>; the definitive DNA textbook, revised in 1994, with a 9 page postscript.</li>
<li><a href="/wiki/Matt_Ridley.html" title="Matt Ridley">Ridley, Matt</a>; <i>Francis Crick: Discoverer of the Genetic Code (Eminent Lives)</i> first published in June 2006 in the USA and then to be in the UK September 2006, by HarperCollins Publishers; 192 pp, <a  class="internal">ISBN 006082333X</a></li>
<li>Watson, James D. and Francis H.C. Crick. <a  class="external text" title="http://www.nature.com/nature/dna50/watsoncrick.pdf">A structure for Deoxyribose Nucleic Acid</a> (PDF). <i><a href="/wiki/Nature_%28journal%29.html" title="Nature (journal)">Nature</a></i> 171, 737 – 738, <a href="/wiki/April_25.html" title="April 25">25 April</a> <a href="/wiki/1953.html" title="1953">1953</a>.</li>
<li>Watson, James D. <i>DNA: The Secret of Life</i> <a  class="internal">ISBN 0375415467</a>.</li>
<li>Watson, James D. <a href="/wiki/The_Double_Helix.html" title="The Double Helix">The Double Helix: A Personal Account of the Discovery of the Structure of DNA (Norton Critical Editions)</a>. <a  class="internal">ISBN 0393950751</a></li>
<li>Chomet, S. (Ed.), DNA Genesis of a Discovery, <i>Newman-Hemisphere Press, London, 1994.</i></li>
<li>Delmonte, C.S. and Mann, L.R.B. <a  class="external text" title="http://www.ias.ac.in/currsci/dec102003/1564.pdf">Variety in DNA tertiary structure</a>. Current Science, 85 (11), 1564 – 1570, 10 December 2003.</li>
<li>Miller, Kenneth R., and Levin, Joseph. <i>Biology</i>. Saddle River, New Jersey: Prentice Hall, 2002.</li>
</ul>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: External links">edit</a>]</div>
<p><a name="External_links" id="External_links"></a></p>
<h2>External links</h2>
<ul>
<li><a  class="external text" title="http://www.dnaftb.org/dnaftb/">DNA from the beginning</a></li>
<li><a  class="external text" title="http://www.dnahack.com/index.html">DNA hack: The website for Amateur Genetic Engineering</a></li>
<li><a  class="external text" title="http://www.packer34.freeserve.co.uk/rememberingfranciscrickacelebration.htm">on Francis Crick</a></li>
<li><a  class="external text" title="http://www.packer34.freeserve.co.uk/selectedTATAwebsites.htm">First press stories on DNA</a></li>
<li><a  class="external text" title="http://en.wikipedia.org/wiki/Image:Rosalindfranklinsjokecard.jpg">'Death' of DNA Helix (Crystaline) joke funeral card</a>.</li>
<li><a  class="external text" title="http://www.nature.com/nature/dna50/archive.html">Double helix: 50 years of DNA</a>, <a href="/wiki/Nature_%28journal%29.html" title="Nature (journal)">Nature</a>.</li>
<li><a  class="external text" title="http://fantsite.narod.ru/articles/37.html">DNA The Program (in Russian)</a>.</li>
<li><a  class="external text" title="http://www.genome.gov/10506367">U.S. National DNA Day</a> Watch videos and participate in real-time chat with top scientists</li>
<li><a  class="external text" title="http://www.genome.gov/10506718">Genetic Education Modules for Teachers</a> <i>DNA from the Beginning</i> Study Guide</li>
<li><a  class="external text" title="http://www.genome.gov/glossary.cfm">Talking Glossary of Genetic Terms</a> In Spanish, too</li>
<li><a  class="external text" title="http://osulibrary.oregonstate.edu/specialcollections/coll/pauling/dna/index.html">Linus Pauling and the Race for DNA</a></li>
<li><a  class="external text" title="http://www.bbc.co.uk/bbcfour/audiointerviews/profilepages/crickwatson1.shtml">Listen to Francis Crick and James Watson talking on the BBC in 1962, 1972, and 1974</a></li>
<li><a  class="external text" title="http://news.bbc.co.uk/1/hi/sci/tech/2949629.stm">17 April, 2003, BBC News: Most ancient DNA ever?</a></li>
<li><a  class="external text" title="http://www.dnakin.com">Using DNA in Genealogical Research</a></li>
<li><a  class="external text" title="http://www.dnai.org">DNA Interactive</a> (requires <a href="/wiki/Macromedia_Flash.html" title="Macromedia Flash">Macromedia Flash</a>)</li>
<li><a  class="external text" title="http://www.rcsb.org/pdb/static.do?p=education_discussion/molecule_of_the_month/pdb23_1.html">DNA: RCSB PDB Molecule of the Month</a></li>
<li><a  class="external text" title="http://www.fidelitysystems.com/Unlinked_DNA.html">DNA under electron microscope</a></li>
<li><a  class="external text" title="http://www.ccrnp.ncifcrf.gov/~toms/LeftHanded.DNA.html">Left-handed DNA Hall of Fame</a></li>
<li><a  class="external text" title="http://www.myfirstbookaboutdna.com">My First Book About DNA</a> Designed for children to learn more about DNA.</li>
<li><a  class="external text" title="http://dmoz.org/Science/Biology/Biochemistry_and_Molecular_Biology/Biomolecules/Nucleic_Acids/">Nucleic Acids</a> at the <a href="/wiki/Open_Directory_Project.html" title="Open Directory Project">Open Directory Project</a></li>
<li><a  class="external text" title="http://www.zytologie-online.net/dna.php">DNA Replication and Translation / Cell Biology</a></li>
<li><a  class="external text" title="http://dnawiz.com/">DNA Articles</a> DNA Articles and Information collected from various sources.</li>
<li><a  class="external text" title="http://www.livescience.com/technology/060620_dna_nanotech.html">Beyond Biology: Making Factories and Computers with DNA</a></li>
<li><a  class="external text" title="http://www.maxanim.com/genetics/Dna%20Replication/Dna%20Replication.htm">DNA Replication</a>Animation of DNA</li>
<li><a  class="external text" title="http://www.dna-worldwide.com/s.nl/it.I/id.23/.f?sc=13&amp;category=26">DNA Testing Guide: Expert Guide written from the feedback of thousands of people who have taken a DNA test</a></li>
</ul>
<p><br clear="all" /></p>
<table align="center" class="toccolours">
<tr bgcolor="pink">
<th colspan="3" align="center"><a href="/wiki/Nucleic_acid.html" title="Nucleic acid">Nucleic acids</a> and <a href="/wiki/Oligonucleotide.html" title="Oligonucleotide">Oligonucleotides</a> <small><small class="editlink noprint plainlinksneverexpand"><a  class="external text" title="http://en.wikipedia.org/w/index.php?title=Template:Nucleic_acids&amp;action=edit">edit</a></small></small></th>
</tr>
<tr>
<th align="right"><b><a href="/wiki/Nucleobase.html" title="Nucleobase">Nucleobases</a></b>:</th>
<td colspan="2"><small><a href="/wiki/Adenine.html" title="Adenine">Adenine</a> | <a href="/wiki/Thymine.html" title="Thymine">Thymine</a> | <a href="/wiki/Uracil.html" title="Uracil">Uracil</a> | <a href="/wiki/Guanine.html" title="Guanine">Guanine</a> | <a href="/wiki/Cytosine.html" title="Cytosine">Cytosine</a> | <a href="/wiki/Purine.html" title="Purine">Purine</a> | <a href="/wiki/Pyrimidine.html" title="Pyrimidine">Pyrimidine</a></small></td>
</tr>
<tr>
<th align="right"><b><a href="/wiki/Nucleoside.html" title="Nucleoside">Nucleosides</a></b>:</th>
<td colspan="2"><small><a href="/wiki/Adenosine.html" title="Adenosine">Adenosine</a> | <a href="/wiki/Uridine.html" title="Uridine">Uridine</a> | <a href="/wiki/Guanosine.html" title="Guanosine">Guanosine</a> | <a href="/wiki/Cytidine.html" title="Cytidine">Cytidine</a> | <a href="/wiki/Deoxyadenosine.html" title="Deoxyadenosine">Deoxyadenosine</a> | <a href="/wiki/Thymidine.html" title="Thymidine">Thymidine</a> | <a href="/wiki/Deoxyguanosine.html" title="Deoxyguanosine">Deoxyguanosine</a> | <a href="/wiki/Deoxycytidine.html" title="Deoxycytidine">Deoxycytidine</a></small></td>
</tr>
<tr>
<th align="right"><b><a href="/wiki/Nucleotide.html" title="Nucleotide">Nucleotides</a></b>:</th>
<td colspan="2"><small><a href="/wiki/Adenosine_monophosphate.html" title="Adenosine monophosphate">AMP</a> | <a href="/wiki/Uridine_monophosphate.html" title="Uridine monophosphate">UMP</a> | <a href="/wiki/Guanosine_monophosphate.html" title="Guanosine monophosphate">GMP</a> | <a href="/wiki/Cytidine_monophosphate.html" title="Cytidine monophosphate">CMP</a> | <a href="/wiki/Adenosine_diphosphate.html" title="Adenosine diphosphate">ADP</a> | <a href="/wiki/Uridine_diphosphate.html" title="Uridine diphosphate">UDP</a> | <a href="/wiki/Guanosine_diphosphate.html" title="Guanosine diphosphate">GDP</a> | <a href="/wiki/Cytidine_diphosphate.html" title="Cytidine diphosphate">CDP</a> | <a href="/wiki/Adenosine_triphosphate.html" title="Adenosine triphosphate">ATP</a> | <a href="/wiki/Uridine_triphosphate.html" title="Uridine triphosphate">UTP</a> | <a href="/wiki/Guanosine_triphosphate.html" title="Guanosine triphosphate">GTP</a> | <a href="/wiki/Cytidine_triphosphate.html" title="Cytidine triphosphate">CTP</a> | <a href="/wiki/Cyclic_adenosine_monophosphate.html" title="Cyclic adenosine monophosphate">cAMP</a> | <a href="/wiki/Cyclic_guanosine_monophosphate.html" title="Cyclic guanosine monophosphate">cGMP</a></small></td>
</tr>
<tr>
<th align="right"><b><a href="/wiki/Deoxynucleotide.html" title="Deoxynucleotide">Deoxynucleotides</a></b>:</th>
<td colspan="2"><small><a  class="new" title="Deoxyadenosine monophosphate">dAMP</a> | <a href="/wiki/Thymidine_monophosphate.html" title="Thymidine monophosphate">TMP</a> | <a  class="new" title="Deoxyguanosine monophosphate">dGMP</a> | <a href="/wiki/Deoxycytidine.html" title="Deoxycytidine">dCMP</a> | <a  class="new" title="Deoxyadenosine diphosphate">dADP</a> | <a href="/wiki/Thymidine_diphosphate.html" title="Thymidine diphosphate">TDP</a> | <a  class="new" title="Deoxyguanosine diphosphate">dGDP</a> | <a  class="new" title="Deoxycytidine diphosphate">dCDP</a> | <a href="/wiki/Deoxyadenosine_triphosphate.html" title="Deoxyadenosine triphosphate">dATP</a> | <a href="/wiki/Thymidine_triphosphate.html" title="Thymidine triphosphate">TTP</a> | <a href="/wiki/Deoxyguanosine_triphosphate.html" title="Deoxyguanosine triphosphate">dGTP</a> | <a href="/wiki/Deoxycytidine_triphosphate.html" title="Deoxycytidine triphosphate">dCTP</a></small></td>
</tr>
<tr>
<th align="right"><b><a href="/wiki/RNA.html" title="RNA">Ribonucleic acids</a></b>:</th>
<td colspan="2"><small><a href="/wiki/RNA.html" title="RNA">RNA</a> | <a href="/wiki/Messenger_RNA.html" title="Messenger RNA">mRNA</a> | <a href="/wiki/Transfer_RNA.html" title="Transfer RNA">tRNA</a> | <a href="/wiki/Ribosomal_RNA.html" title="Ribosomal RNA">rRNA</a> | <a href="/wiki/Non-coding_RNA.html" title="Non-coding RNA">ncRNA</a> | <a  class="new" title="Small-guide RNA">sgRNA</a> | <a href="/wiki/Small_hairpin_RNA.html" title="Small hairpin RNA">shRNA</a> | <a href="/wiki/Small_interfering_RNA.html" title="Small interfering RNA">siRNA</a> | <a href="/wiki/Small_nuclear_RNA.html" title="Small nuclear RNA">snRNA</a> | <a href="/wiki/Micro_RNA.html" title="Micro RNA">miRNA</a> | <a href="/wiki/Small_nucleolar_RNA.html" title="Small nucleolar RNA">snoRNA</a> | <a href="/wiki/Locked_nucleic_acid.html" title="Locked nucleic acid">LNA</a></small></td>
</tr>
<tr>
<th align="right"><b><strong class="selflink">Deoxyribonucleic acids</strong></b>:</th>
<td colspan="2"><small><strong class="selflink">DNA</strong> | <a href="/wiki/Mitochondrial_DNA.html" title="Mitochondrial DNA">mtDNA</a> | <a href="/wiki/Complementary_DNA.html" title="Complementary DNA">cDNA</a></small></td>
</tr>
<tr>
<th align="right"><b>Other <a href="/wiki/Nucleic_acid.html" title="Nucleic acid">nucleic acids</a></b>:</th>
<td colspan="2"><small><a href="/wiki/GNA_%28nucleic_acid%29.html" title="GNA (nucleic acid)">GNA</a> | <a href="/wiki/PNA.html" title="PNA">PNA</a> | <a href="/wiki/TNA_%28nucleic_acid%29.html" title="TNA (nucleic acid)">TNA</a></small></td>
</tr>
<tr bgcolor="#CCCCFF">
<td align="left"><a  title="Template:AminoAcids">←Amino acids</a></td>
<td align="center"><a  title="Template:Biochemical families">Major families of biochemicals</a></td>
<td width="10%" align="right"><a  title="Template:Carbohydrates">Carbohydrates→</a></td>
</tr>
</table>
<p><span class="FA" id="de" style="display:none;"></span> <span class="FA" id="nl" style="display:none;"></span></p>

<!-- 
Pre-expand include size: 29043 bytes
Post-expand include size: 7948 bytes
Template argument size: 4060 bytes
Maximum: 2048000 bytes
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:7955-0!1!0!default!!en!2 and timestamp 20060910160511 -->
<div class="printfooter">
Retrieved from "<a </div>
			<div id="catlinks"><p class='catlinks'><a  title="Special:Categories">Categories</a>: <span dir='ltr'><a  title="Category:Articles with unsourced statements">Articles with unsourced statements</a></span> | <span dir='ltr'><a  title="Category:DNA">DNA</a></span> | <span dir='ltr'><a  title="Category:Genetics">Genetics</a></span></p></div>			<!-- end content -->
			<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<ul>
				 <li id="ca-nstab-main" class="selected"><a href="/wiki/DNA.html">Article</a></li>
				 <li id="ca-talk"><a >Discussion</a></li>
				 <li id="ca-edit"><a >Edit this page</a></li>
				 <li id="ca-history"><a >History</a></li>
		</ul>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a >Sign in / create account</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(/images/wiki-en.png);" href="/wiki/Main_Page.html" title="Main Page"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="/wiki/Main_Page.html">Main Page</a></li>
				<li id="n-portal"><a >Community Portal</a></li>
				<li id="n-Featured-articles"><a >Featured articles</a></li>
				<li id="n-currentevents"><a >Current events</a></li>
				<li id="n-recentchanges"><a >Recent changes</a></li>
				<li id="n-randompage"><a >Random article</a></li>
				<li id="n-help"><a >Help</a></li>
				<li id="n-contact"><a >Contact Wikipedia</a></li>
				<li id="n-sitesupport"><a >Donations</a></li>
			</ul>
		</div>
	</div>
		<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="/wiki/Special:Search" id="searchform"><div>
				<input id="searchInput" name="search" type="text" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" value="Search" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a >What links here</a></li>
				<li id="t-recentchangeslinked"><a >Related changes</a></li>
<li id="t-upload"><a >Upload file</a></li>
<li id="t-specialpages"><a >Special pages</a></li>
				<li id="t-print"><a >Printable version</a></li>				<li id="t-permalink"><a >Permanent link</a></li><li id="t-cite"><a >Cite this article</a></li>			</ul>
		</div>
	</div>
	<div id="p-lang" class="portlet">
		<h5>In other languages</h5>
		<div class="pBody">
			<ul>
				<li class="interwiki-ar"><a >العربية</a></li>
				<li class="interwiki-zh-min-nan"><a >Bân-lâm-gú</a></li>
				<li class="interwiki-bs"><a >Bosanski</a></li>
				<li class="interwiki-bg"><a >Български</a></li>
				<li class="interwiki-ca"><a >Català</a></li>
				<li class="interwiki-cs"><a >Česky</a></li>
				<li class="interwiki-cy"><a >Cymraeg</a></li>
				<li class="interwiki-da"><a >Dansk</a></li>
				<li class="interwiki-pdc"><a >Deitsch</a></li>
				<li class="interwiki-de"><a >Deutsch</a></li>
				<li class="interwiki-et"><a >Eesti</a></li>
				<li class="interwiki-el"><a >Ελληνικά</a></li>
				<li class="interwiki-es"><a >Español</a></li>
				<li class="interwiki-eo"><a >Esperanto</a></li>
				<li class="interwiki-eu"><a >Euskara</a></li>
				<li class="interwiki-fr"><a >Français</a></li>
				<li class="interwiki-ga"><a >Gaeilge</a></li>
				<li class="interwiki-gl"><a >Galego</a></li>
				<li class="interwiki-ko"><a >한국어</a></li>
				<li class="interwiki-id"><a >Bahasa Indonesia</a></li>
				<li class="interwiki-it"><a >Italiano</a></li>
				<li class="interwiki-he"><a >עברית</a></li>
				<li class="interwiki-ht"><a >Krèyol ayisyen</a></li>
				<li class="interwiki-la"><a >Latina</a></li>
				<li class="interwiki-lv"><a >Latviešu</a></li>
				<li class="interwiki-lt"><a >Lietuvių</a></li>
				<li class="interwiki-hu"><a >Magyar</a></li>
				<li class="interwiki-mk"><a >Македонски</a></li>
				<li class="interwiki-ms"><a >Bahasa Melayu</a></li>
				<li class="interwiki-nl"><a >Nederlands</a></li>
				<li class="interwiki-ja"><a >日本語</a></li>
				<li class="interwiki-no"><a >Norsk (bokmål)</a></li>
				<li class="interwiki-nn"><a >Norsk (nynorsk)</a></li>
				<li class="interwiki-pam"><a >Pampangan</a></li>
				<li class="interwiki-pl"><a >Polski</a></li>
				<li class="interwiki-pt"><a >Português</a></li>
				<li class="interwiki-ro"><a >Română</a></li>
				<li class="interwiki-ru"><a >Русский</a></li>
				<li class="interwiki-sq"><a >Shqip</a></li>
				<li class="interwiki-simple"><a >Simple English</a></li>
				<li class="interwiki-sk"><a >Slovenčina</a></li>
				<li class="interwiki-sl"><a >Slovenščina</a></li>
				<li class="interwiki-sr"><a >Српски / Srpski</a></li>
				<li class="interwiki-sh"><a >Srpskohrvatski / Српскохрватски</a></li>
				<li class="interwiki-su"><a >Basa Sunda</a></li>
				<li class="interwiki-fi"><a >Suomi</a></li>
				<li class="interwiki-sv"><a >Svenska</a></li>
				<li class="interwiki-tl"><a >Tagalog</a></li>
				<li class="interwiki-ta"><a >தமிழ்</a></li>
				<li class="interwiki-th"><a >ไทย</a></li>
				<li class="interwiki-vi"><a >Tiếng Việt</a></li>
				<li class="interwiki-tr"><a >Türkçe</a></li>
				<li class="interwiki-uk"><a >Українська</a></li>
				<li class="interwiki-ur"><a >اردو</a></li>
				<li class="interwiki-zh"><a >中文</a></li>
			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a ><img src="/skins-1.5/common/images/poweredby_mediawiki_88x31.png" alt="MediaWiki" /></a></div>
				<div id="f-copyrightico"><a ><img src="/images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>
			<ul id="f-list">
				<li id="lastmod"> This page was last modified 09:59, 10 September 2006.</li>
				<li id="copyright">All text is available under the terms of the <a class='internal'  title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal'  title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the Wikimedia Foundation, Inc.<br /></li>
				<li id="privacy"><a  title="wikimedia:Privacy policy">Privacy policy</a></li>
				<li id="about"><a  title="Wikipedia:About">About Wikipedia</a></li>
				<li id="disclaimer"><a  title="Wikipedia:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
		
	
		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
</div>
<!-- Served by srv19 in 0.081 secs. --></body></html>
