<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
		<meta name="keywords" content="Enzyme,1700s,1800s,1897,ATPase,Acetylcholinesterase,Activation energy,Active site,Active transport,Adenosine triphosphate,Agriculture" />
<link rel="shortcut icon"  />
<link rel="search" type="application/opensearchdescription+xml"  />
<link rel="copyright"  />
		<title>Enzyme - Wikipedia, the free encyclopedia</title>
		<style type="text/css" media="screen,projection">/*<![CDATA[*/ @import "/skins-1.5/monobook/main.css?9"; /*]]>*/</style>
		<link rel="stylesheet" type="text/css" media="print"  />
		<!--[if lt IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE50Fixes.css";</style><![endif]-->
		<!--[if IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE55Fixes.css";</style><![endif]-->
		<!--[if IE 6]><style type="text/css">@import "/skins-1.5/monobook/IE60Fixes.css";</style><![endif]-->
		<!--[if IE 7]><style type="text/css">@import "/skins-1.5/monobook/IE70Fixes.css?1";</style><![endif]-->
		<!--[if lt IE 7]><script type="text/javascript" src="/skins-1.5/common/IEFixes.js"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
		
		<script type= "text/javascript">
			var skin = "monobook";
			var stylepath = "/skins-1.5";

			var wgArticlePath = "/wiki/$1";
			var wgScriptPath = "/w";
			var wgServer = "http://en.wikipedia.org";
                        
			var wgCanonicalNamespace = "";
			var wgNamespaceNumber = 0;
			var wgPageName = "Enzyme";
			var wgTitle = "Enzyme";
			var wgArticleId = 9257;
			var wgIsArticle = true;
                        
			var wgUserName = null;
			var wgUserLanguage = "en";
			var wgContentLanguage = "en";
		</script>
		                
		<script type="text/javascript" src="/skins-1.5/common/wikibits.js?1"><!-- wikibits js --></script>
		<script type="text/javascript" src="/w/index.php?title=-&amp;action=raw&amp;gen=js"><!-- site js --></script>
		<style type="text/css">/*<![CDATA[*/
@import "/w/index.php?title=MediaWiki:Common.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=MediaWiki:Monobook.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=-&action=raw&gen=css&maxage=2678400";
/*]]>*/</style>
		<!-- Head Scripts -->
		<script type="text/javascript" src="/skins-1.5/common/ajax.js"></script>
	</head>
<body  class="mediawiki ns-0 ltr">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
		<div id="siteNotice"><div style="text-align:right; font-size:80%">Your <b><a  class="extiw" title="wikimedia:Fundraising">continued donations</a></b> keep Wikipedia running!&nbsp;&nbsp;&nbsp;&nbsp;</div>
</div>		<h1 class="firstHeading">Enzyme</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a >navigation</a>, <a >search</a></div>			<!-- start content -->
			<div class="thumb tright">
<div style="width:312px;"><a  class="internal" title="Ribbon diagram of the enzyme TIM. TIM is catalytically perfect, meaning its conversion rate is limited, or nearly limited, to its substrate diffusion rate."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/8/81/Triosephosphate_isomerase.jpg/310px-Triosephosphate_isomerase.jpg" alt="Ribbon diagram of the enzyme TIM. TIM is catalytically perfect, meaning its conversion rate is limited, or nearly limited, to its substrate diffusion rate." width="310" height="310" longdesc="/wiki/Image:Triosephosphate_isomerase.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Ribbon diagram of the enzyme <a href="/wiki/Triosephosphateisomerase.html" title="Triosephosphateisomerase">TIM</a>. TIM is <b><a href="/wiki/Catalytically_perfect_enzyme.html" title="Catalytically perfect enzyme">catalytically perfect</a></b>, meaning its conversion rate is limited, or nearly limited, to its substrate diffusion rate.</div>
</div>
</div>
<p><b>Enzymes</b> are <a href="/wiki/Protein.html" title="Protein">proteins</a> that accelerate, or <a href="/wiki/Catalyze.html" title="Catalyze">catalyze</a>, <a href="/wiki/Chemical_reaction.html" title="Chemical reaction">chemical reactions</a>. In these reactions, the molecules at the beginning of the process are called <a href="/wiki/Substrate_%28biochemistry%29.html" title="Substrate (biochemistry)">substrates</a> and the enzyme converts these into different molecules: the products. Almost all processes in the <a href="/wiki/Cell_%28biology%29.html" title="Cell (biology)">cell</a> need enzymes in order to occur at significant rates. Consequently, since enzymes are extremely selective for their substrates and speed up only a few reactions from among many possibilities, the set of enzymes made in a cell determines which <a href="/wiki/Metabolic_pathway.html" title="Metabolic pathway">metabolic pathways</a> occur in that cell.</p>
<p>Enzymes are known to catalyze about 4,000 reactions.<sup id="_ref-0" class="reference"><a  title="">[1]</a></sup> However, not all biological catalysts are proteins, since some <a href="/wiki/RNA.html" title="RNA">RNA</a> <a href="/wiki/Molecule.html" title="Molecule">molecules</a> called <a href="/wiki/Ribozyme.html" title="Ribozyme">ribozymes</a> can also catalyze reactions. Enzymes are usually named according to the reaction they catalyze. Typically the suffix <i>-ase</i> is added to the name of the <a href="/wiki/Substrate_%28biochemistry%29.html" title="Substrate (biochemistry)">substrate</a> (<i>e.g.</i>, <a href="/wiki/Lactase.html" title="Lactase">lactase</a> is the enzyme that cleaves <a href="/wiki/Lactose.html" title="Lactose">lactose</a>) or the type of reaction (<i>e.g.</i>, <a href="/wiki/DNA_polymerase.html" title="DNA polymerase">DNA polymerase</a> forms DNA polymers).</p>
<p>Like all catalysts, enzymes work by providing an alternative <a href="/wiki/Reaction_coordinate.html" title="Reaction coordinate">path</a> of lower <a href="/wiki/Activation_energy.html" title="Activation energy">activation energy</a> for a reaction and dramatically accelerating its rate. Some enzymes can make their conversion of substrate to product occur many millions of times faster. For example, the reaction catalysed by <a href="/wiki/Orotidine_5%27-phosphate_decarboxylase.html" title="Orotidine 5'-phosphate decarboxylase">orotidine 5'-phosphate decarboxylase</a> will consume half of its substrate in 78 million years if no enzyme is present. However, when the decarboxylase is added, the same process takes just 25 milliseconds.<sup id="_ref-1" class="reference"><a  title="">[2]</a></sup> Chemically, enzymes are like any catalyst and are not consumed in chemical reactions, nor do they alter the <a href="/wiki/Chemical_equilibrium.html" title="Chemical equilibrium">equilibrium</a> of a reaction. However, enzymes do differ from most other catalysts by being much more specific.</p>
<p>Enzyme activity can be affected by other molecules. <a href="/wiki/Enzyme_inhibitor.html" title="Enzyme inhibitor">Inhibitors</a> are molecules that decrease enzyme activity, and activators are molecules that increase activity. <a href="/wiki/Drug.html" title="Drug">Drugs</a> and <a href="/wiki/Poison.html" title="Poison">poisons</a> are often enzyme inhibitors.</p>
<p>Some enzymes are used commercially, for example, in the synthesis of <a href="/wiki/Antibiotics.html" title="Antibiotics">antibiotics</a>. In addition, some household cleaning products use enzymes to speed up chemical reactions (<i>e.g.</i>, enzymes in biological washing powders break down protein or fat stains on clothes).</p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a ><span class="tocnumber">1</span> <span class="toctext">Etymology and history</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">2</span> <span class="toctext">Structures and mechanisms</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">2.1</span> <span class="toctext">Specificity</span></a>
<ul>
<li class="toclevel-3"><a ><span class="tocnumber">2.1.1</span> <span class="toctext">"Lock and key" model</span></a></li>
<li class="toclevel-3"><a ><span class="tocnumber">2.1.2</span> <span class="toctext">Induced fit model</span></a></li>
</ul>
</li>
<li class="toclevel-2"><a ><span class="tocnumber">2.2</span> <span class="toctext">Allosteric modulation</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">3</span> <span class="toctext">Cofactors and coenzymes</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">3.1</span> <span class="toctext">Cofactors</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">3.2</span> <span class="toctext">Coenzymes</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">4</span> <span class="toctext">Thermodynamics</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">5</span> <span class="toctext">Kinetics</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">6</span> <span class="toctext">Inhibition</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">6.1</span> <span class="toctext">Reversible inhibitors</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">6.2</span> <span class="toctext">Irreversible inhibitors</span></a></li>
<li class="toclevel-2"><a ><span class="tocnumber">6.3</span> <span class="toctext">Uses of inhibitors</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">7</span> <span class="toctext">Biological function</span></a>
<ul>
<li class="toclevel-2"><a ><span class="tocnumber">7.1</span> <span class="toctext">Metabolism</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a ><span class="tocnumber">8</span> <span class="toctext">Control of activity</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">9</span> <span class="toctext">Involvement in disease</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">10</span> <span class="toctext">Naming conventions</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">11</span> <span class="toctext">Industrial applications</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">12</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">13</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">14</span> <span class="toctext">Further reading</span></a></li>
<li class="toclevel-1"><a ><span class="tocnumber">15</span> <span class="toctext">External links</span></a></li>
</ul>
</td>
</tr>
</table>
<p><script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script></p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Etymology and history">edit</a>]</div>
<p><a name="Etymology_and_history" id="Etymology_and_history"></a></p>
<h2>Etymology and history</h2>
<div class="thumb tright">
<div style="width:177px;"><a  class="internal" title="Eduard Buchner"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/b/b2/Eduardbuchner.jpg/175px-Eduardbuchner.jpg" alt="Eduard Buchner" width="175" height="245" longdesc="/wiki/Image:Eduardbuchner.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
<a href="/wiki/Eduard_Buchner.html" title="Eduard Buchner">Eduard Buchner</a></div>
</div>
</div>
<p>As early as the late <a href="/wiki/1700s.html" title="1700s">1700s</a> and early <a href="/wiki/1800s.html" title="1800s">1800s</a>, the digestion of <a href="/wiki/Meat.html" title="Meat">meat</a> by stomach secretions and the conversion of starch to sugars by plant extracts and saliva were known. However, the mechanism by which this occurred was unknown.<sup id="_ref-2" class="reference"><a  title="">[3]</a></sup></p>
<p>In the 19th century, when studying the <a href="/wiki/Fermentation.html" title="Fermentation">fermentation</a> of sugar to alcohol by yeast, <a href="/wiki/Louis_Pasteur.html" title="Louis Pasteur">Louis Pasteur</a> came to the conclusion that this fermentation was catalyzed by a vital force contained within the yeast cells called "<a href="/wiki/Vitalism.html" title="Vitalism">ferments</a>", which were thought to function only within living organisms. He wrote that "alcoholic fermentation is an act correlated with the life and organisation of the yeast cells, not with the death or putrefaction of the cells."<sup id="_ref-3" class="reference"><a  title="">[4]</a></sup></p>
<p>In 1878 German physiologist <a href="/wiki/Wilhelm_K%C3%BChne.html" title="Wilhelm Kühne">Wilhelm Kühne</a> (1837–1900) coined the term <i><a  class="extiw" title="wiktionary:enzyme">enzyme</a></i>, which comes from <a href="/wiki/Greek_language.html" title="Greek language">Greek</a> <i>ενζυμον</i> "in leaven", to describe this process. The word <i>enzyme</i> was used later to refer to nonliving substances such as <a href="/wiki/Pepsin.html" title="Pepsin">pepsin</a>, and the word <i>ferment</i> was used to refer to chemical activity produced by living organisms.</p>
<p>In <a href="/wiki/1897.html" title="1897">1897</a> <a href="/wiki/Eduard_Buchner.html" title="Eduard Buchner">Eduard Buchner</a> began to study the ability of yeast extracts to ferment sugar despite the absence of living yeast cells. In a series of experiments at the <a href="/wiki/Humboldt_University_of_Berlin.html" title="Humboldt University of Berlin">University of Berlin</a>, he found that the sugar was fermented even when there were no living yeast cells in the mixture.<sup id="_ref-4" class="reference"><a  title="">[5]</a></sup> He named the enzyme that brought about the fermentation of sucrose "<a href="/wiki/Zymase.html" title="Zymase">zymase</a>".<sup id="_ref-5" class="reference"><a  title="">[6]</a></sup> In 1907 he received the <a href="/wiki/Nobel_Prize_in_Chemistry.html" title="Nobel Prize in Chemistry">Nobel Prize in Chemistry</a> "for his biochemical research and his discovery of cell-free fermentation".</p>
<p>It was not until 1926, however, that pepsin became the first enzyme to be obtained in pure form by <a href="/wiki/John_Howard_Northrop.html" title="John Howard Northrop">Northrop</a>, <a href="/wiki/James_B._Sumner.html" title="James B. Sumner">Sumner</a> and <a href="/wiki/Wendell_Meredith_Stanley.html" title="Wendell Meredith Stanley">Stanley</a>. Their successful purification and crystallization of pepsin proved that enzymes were proteins, and they were awarded the 1946 Nobel prize in Chemistry.<sup id="_ref-6" class="reference"><a  title="">[7]</a></sup></p>
<p>This discovery that enzymes could be crystalised eventually allowed their structures to be solved by <a href="/wiki/X-ray_crystallography.html" title="X-ray crystallography">x-ray crystallography</a>. This was first done for <a href="/wiki/Lysozyme.html" title="Lysozyme">lysozyme</a>, an enzyme found in tears, saliva and <a href="/wiki/Egg_white.html" title="Egg white">egg whites</a> that digests the coating of some bacteria; the structure was solved by a group led by <a href="/wiki/David_Chilton_Phillips.html" title="David Chilton Phillips">David Chilton Phillips</a> and published in 1965.<sup id="_ref-7" class="reference"><a  title="">[8]</a></sup> This high-resolution structure of lysozyme marked the beginning of the field of <a href="/wiki/Structural_biology.html" title="Structural biology">structural biology</a> and the effort to understand how enzymes work at an atomic level of detail.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Structures and mechanisms">edit</a>]</div>
<p><a name="Structures_and_mechanisms" id="Structures_and_mechanisms"></a></p>
<h2>Structures and mechanisms</h2>
<div class="thumb tright">
<div style="width:302px;"><a  class="internal" title="Ribbon-diagram showing the active sites of Carbonic anhydrase. The grey spheres are the zinc ions in the four active sites of this carbonic anhydrase enzyme and are held within two protein chains. Diagram drawn from PDB 1DDZ."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/a/a8/Carbonic_anhydrase.jpg/300px-Carbonic_anhydrase.jpg" alt="Ribbon-diagram showing the active sites of Carbonic anhydrase. The grey spheres are the zinc ions in the four active sites of this carbonic anhydrase enzyme and are held within two protein chains. Diagram drawn from PDB 1DDZ." width="300" height="300" longdesc="/wiki/Image:Carbonic_anhydrase.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Ribbon-diagram showing the active sites of Carbonic anhydrase. The grey spheres are the zinc ions in the four active sites of this <a href="/wiki/Carbonic_anhydrase.html" title="Carbonic anhydrase">carbonic anhydrase</a> enzyme and are held within two protein chains. Diagram drawn from <a  class="external text" title="http://www.rcsb.org/pdb/explore.do?structureId=1DDZ">PDB 1DDZ</a>.</div>
</div>
</div>
<p>The activities of enzymes are determined by their <a href="/wiki/Quaternary_structure.html" title="Quaternary structure">three-dimensional structure</a>.<sup id="_ref-8" class="reference"><a  title="">[9]</a></sup></p>
<p>Most enzymes are much larger than the substrates they act on, and only a very small portion of the enzyme (around 3–4 <a href="/wiki/Amino_acid.html" title="Amino acid">amino acids</a>) is directly involved in catalysis.<sup id="_ref-9" class="reference"><a  title="">[10]</a></sup> The region that contains these catalytic residues and binds the substrate and then carries out the reaction is known as the <a href="/wiki/Active_site.html" title="Active site">active site</a>. Some enzymes also contain sites that bind <a href="/wiki/Cofactor_%28biochemistry%29.html" title="Cofactor (biochemistry)">cofactors</a>, which are needed for catalysis. Some enzymes also have binding sites for small molecules, which are often direct or <a  title="">indirect</a> products or substrates of the reaction catalyzed. This binding can serve to increase or decrease the enzyme's activity, providing a means for <a href="/wiki/Feedback.html" title="Feedback">feedback</a> regulation.</p>
<p>Like all proteins, enzymes are made as long, linear chains of amino acids that <a href="/wiki/Protein_folding.html" title="Protein folding">fold</a> to produce a <a href="/wiki/Tertiary_structure.html" title="Tertiary structure">three-dimensional product</a>. Each unique amino acid sequence produces a unique structure, which has unique properties. Individual protein chains may sometimes group together to form a <a href="/wiki/Protein_complex.html" title="Protein complex">protein complex</a>. Most enzymes can be <a href="/wiki/Denaturation_%28biochemistry%29.html" title="Denaturation (biochemistry)">denatured</a>—that is, unfolded and inactivated—by heating, which destroys the <a href="/wiki/Tertiary_structure.html" title="Tertiary structure">three-dimensional structure</a> of the protein. Depending on the enzyme, denaturation may be reversible or irreversible.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Specificity">edit</a>]</div>
<p><a name="Specificity" id="Specificity"></a></p>
<h3>Specificity</h3>
<p>Enzymes are usually specific as to which reactions they catalyze and the <a href="/wiki/Substrate_%28biochemistry%29.html" title="Substrate (biochemistry)">substrates</a> that are involved in these reactions. Shape, charge complementarity, and <a href="/wiki/Hydrophilic.html" title="Hydrophilic">hydrophilic</a>/<a href="/wiki/Hydrophobic.html" title="Hydrophobic">hydrophobic</a> characters of enzymes and substrates are responsible for this specificity. Enzymes can also show impressive levels of <a href="/wiki/Stereospecificity.html" title="Stereospecificity">stereospecificity</a>, <a href="/wiki/Regioselectivity.html" title="Regioselectivity">regioselectivity</a> and <a href="/wiki/Chemoselectivity.html" title="Chemoselectivity">chemoselectivity</a>.<sup id="_ref-10" class="reference"><a  title="">[11]</a></sup></p>
<p>Some of the enzymes showing the highest specificity and accuracy are involved in the copying and expression of the <a href="/wiki/Genome.html" title="Genome">genome</a>. These enzymes have "proof-reading" mechanisms. Here, an enzyme such as <a href="/wiki/DNA_polymerase.html" title="DNA polymerase">DNA polymerase</a> catalyses a reaction in a first step and then checks the product is correct in a second step.<sup id="_ref-11" class="reference"><a  title="">[12]</a></sup> This two-step process results in average error rates of less than one error 1 in 100 million reactions in high-fidelity mammalian polymerases.<sup id="_ref-12" class="reference"><a  title="">[13]</a></sup> Similar proofreading mechanisms are also found in <a href="/wiki/Aminoacyl_tRNA_synthetase.html" title="Aminoacyl tRNA synthetase">aminoacyl tRNA synthetases</a><sup id="_ref-13" class="reference"><a  title="">[14]</a></sup> and <a href="/wiki/Ribosome.html" title="Ribosome">ribosomes</a>.<sup id="_ref-14" class="reference"><a  title="">[15]</a></sup></p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: &quot;Lock and key&quot; model">edit</a>]</div>
<p><a name=".22Lock_and_key.22_model"></a></p>
<h4>"Lock and key" model</h4>
<div class="thumb tright">
<div style="width:352px;"><a  class="internal" title="Diagrams to show the induced fit hypothesis of enzyme action."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/d/dc/Two_substrates_b.svg/350px-Two_substrates_b.svg.png" alt="Diagrams to show the induced fit hypothesis of enzyme action." width="350" height="137" longdesc="/wiki/Image:Two_substrates_b.svg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Diagrams to show the induced fit hypothesis of enzyme action.</div>
</div>
</div>
<p>Enzymes are very specific, and it was suggested by <a href="/wiki/Emil_Fischer.html" title="Emil Fischer">Emil Fischer</a> in 1894 that this was because both the enzyme and the substrate possess specific complementary geometric shapes that fit exactly into one another.<sup id="_ref-15" class="reference"><a  title="">[16]</a></sup> This is often referred to as "the lock and key" model. An enzyme combines with its substrate(s) to form a short-lived enzyme-substrate complex. However, while this model explains enzyme specificity, it fails to explain the stabilization of the transition state which occurs.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Induced fit model">edit</a>]</div>
<p><a name="Induced_fit_model" id="Induced_fit_model"></a></p>
<h4>Induced fit model</h4>
<p>In 1958 <a  class="new" title="Daniel Koshland">Daniel Koshland</a> suggested a modification to the "lock and key" model.<sup id="_ref-16" class="reference"><a  title="">[17]</a></sup> Since enzymes are rather flexible structures, the active site can be modified as the substrate interacts with the enzyme. As a result, the amino acid <a href="/wiki/Side_chain.html" title="Side chain">side chains</a> which make up the active site are molded into a precise shape which enables the enzyme to perform its catalytic function. In some cases the substrate molecule also changes shape slightly as it enters the active site. Unlike the "Lock and key" model, this model explains both enzyme specificity and the stabilization of the transition state.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Allosteric modulation">edit</a>]</div>
<p><a name="Allosteric_modulation" id="Allosteric_modulation"></a></p>
<h3>Allosteric modulation</h3>
<p><a href="/wiki/Allosteric.html" title="Allosteric">Allosteric</a> enzymes change their structure in response to binding of <a href="/wiki/Effector_%28biology%29.html" title="Effector (biology)">effectors</a>. Modulation can be direct, where the effector binds directly to <a href="/wiki/Binding_site.html" title="Binding site">binding sites</a> in the enzyme, or indirect, where the effector binds to other proteins or <a href="/wiki/Protein_subunit.html" title="Protein subunit">protein subunits</a> that interact with the allosteric enzyme and thus influence catalytic activity.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Cofactors and coenzymes">edit</a>]</div>
<p><a name="Cofactors_and_coenzymes" id="Cofactors_and_coenzymes"></a></p>
<h2>Cofactors and coenzymes</h2>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Cofactors">edit</a>]</div>
<p><a name="Cofactors" id="Cofactors"></a></p>
<h3>Cofactors</h3>
<p>Some enzymes do not need any additional components to show full activity. However, others require non-protein molecules to be bound for activity. Cofactors can be either <a href="/wiki/Inorganic.html" title="Inorganic">inorganic</a> (<i>e.g.</i>, metal ions and <a href="/wiki/Iron-sulfur_cluster.html" title="Iron-sulfur cluster">iron-sulfur clusters</a>) or <a href="/wiki/Organic_molecules.html" title="Organic molecules">organic compounds</a>, (e.g., <a href="/wiki/Flavin.html" title="Flavin">flavin</a> and <a href="/wiki/Heme.html" title="Heme">heme</a>). Organic cofactors are usually called <a href="/wiki/Prosthetic_groups.html" title="Prosthetic groups">prosthetic groups</a>. Tightly-bound cofactors are distinguished from <a href="/wiki/Coenzymes.html" title="Coenzymes">coenzymes</a>, such as <a href="/wiki/Nicotinamide_adenine_dinucleotide.html" title="Nicotinamide adenine dinucleotide">NADH</a>, since cofactors are regenerated as part of the catalytic mechanism and are not released from the active site during the reaction.</p>
<p>An example of an enzyme that contains a cofactor is <a href="/wiki/Carbonic_anhydrase.html" title="Carbonic anhydrase">carbonic anhydrase</a>, and is shown in the diagram above with four zinc cofactors bound in its active sites.<sup id="_ref-17" class="reference"><a  title="">[18]</a></sup> These tightly-bound molecules are usually found in the active site and are involved in catalysis. For example, flavin and heme cofactors are often involved in <a href="/wiki/Redox.html" title="Redox">redox</a> reactions.</p>
<p>Enzymes that require a cofactor but do not have one bound are called <a href="/wiki/Apoenzyme.html" title="Apoenzyme">apoenzymes</a>. An apoenzyme together with its cofactor(s) is called a <a href="/wiki/Holoenzyme.html" title="Holoenzyme">holoenzyme</a> (<i>i.e.</i>, the active form). Most cofactors are not covalently attached to an enzyme, but are very tightly bound. However, organic prosthetic groups can be covalently bound (<i>e.g.</i>, <a href="/wiki/Thiamine_pyrophosphate.html" title="Thiamine pyrophosphate">thiamine pyrophosphate</a> in the enzyme <a href="/wiki/Pyruvate_dehydrogenase.html" title="Pyruvate dehydrogenase">pyruvate dehydrogenase</a>).</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Coenzymes">edit</a>]</div>
<p><a name="Coenzymes" id="Coenzymes"></a></p>
<h3>Coenzymes</h3>
<div class="thumb tleft">
<div style="width:152px;"><a  class="internal" title="Space-filling model of the coenzyme NADH"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/e/ed/NADH-3D-vdW.png/150px-NADH-3D-vdW.png" alt="Space-filling model of the coenzyme NADH" width="150" height="167" longdesc="/wiki/Image:NADH-3D-vdW.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Space-filling model of the coenzyme NADH</div>
</div>
</div>
<p>Coenzymes are small molecules that transport chemical groups from one enzyme to another.<sup id="_ref-18" class="reference"><a  title="">[19]</a></sup> Some of these chemicals such as <a href="/wiki/Riboflavin.html" title="Riboflavin">riboflavin</a>, <a href="/wiki/Thiamine.html" title="Thiamine">thiamine</a> and <a href="/wiki/Folic_acid.html" title="Folic acid">folic acid</a> are <a href="/wiki/Vitamins.html" title="Vitamins">vitamins</a>, this is when these compounds cannot be made in the body and must be acquired from the diet. The chemical groups carried include the hydride ion (H+ + 2e-) carried by <a href="/wiki/Nicotinamide_adenine_dinucleotide.html" title="Nicotinamide adenine dinucleotide">NAD or NADP<sup>+</sup></a>, the acetyl group carried by <a href="/wiki/Coenzyme_A.html" title="Coenzyme A">coenzyme A</a>, formyl, methenyl or methyl groups carried by <a href="/wiki/Folic_acid.html" title="Folic acid">folic acid</a> and the methyl group carried by <a href="/wiki/S-adenosylmethionine.html" title="S-adenosylmethionine">S-adenosylmethionine</a>.</p>
<p>Since coenzymes are chemically changed as a consequence of enzyme action, it is useful to consider coenzymes to be a special class of substrates, or second substrates, which are common to many different enzymes. For example, about 700 enzymes are known to use the cofactor NADH.<sup id="_ref-19" class="reference"><a  title="">[20]</a></sup></p>
<p>Coenzymes are usually regenerated and their concentrations maintained at a steady level inside the cell: for example, NADPH is regenerated through the <a href="/wiki/Pentose_phosphate_pathway.html" title="Pentose phosphate pathway">pentose phosphate pathway</a> and <i>S</i>-adenosylmethionine by methionine adenosyltransferase.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Thermodynamics">edit</a>]</div>
<p><a name="Thermodynamics" id="Thermodynamics"></a></p>
<h2>Thermodynamics</h2>
<dl>
<dd>
<div class="noprint"><i>Main articles: <a href="/wiki/Activation_energy.html" title="Activation energy">Activation energy</a>, <a href="/wiki/Thermodynamic_equilibrium.html" title="Thermodynamic equilibrium">Thermodynamic equilibrium</a>, and <a href="/wiki/Chemical_equilibrium.html" title="Chemical equilibrium">Chemical equilibrium</a></i></div>
</dd>
</dl>
<div class="thumb tright">
<div style="width:302px;"><a  class="internal" title="Diagram of a catalytic reaction, showing the energy niveau at each stage of the reaction. The substrates usually need a large amount of energy to reach the transition state, which then decays into the end product. The enzyme stabilizes the transition state, reducing the energy needed to form this species and thus reducing the energy required to form products."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/d/df/Activation2.svg/300px-Activation2.svg.png" alt="Diagram of a catalytic reaction, showing the energy niveau at each stage of the reaction. The substrates usually need a large amount of energy to reach the transition state, which then decays into the end product. The enzyme stabilizes the transition state, reducing the energy needed to form this species and thus reducing the energy required to form products." width="300" height="235" longdesc="/wiki/Image:Activation2.svg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Diagram of a catalytic reaction, showing the energy <i>niveau</i> at each stage of the reaction. The substrates usually need a large amount of energy to reach the transition state, which then decays into the end product. The enzyme stabilizes the transition state, reducing the energy needed to form this species and thus reducing the energy required to form products.</div>
</div>
</div>
<p>As with all catalysts, all reactions catalyzed by enzymes must be "spontaneous" (containing a net negative <a href="/wiki/Gibbs_free_energy.html" title="Gibbs free energy">Gibbs free energy</a>). In the presence of an enzyme, a reaction runs in the same direction as it would without the enzyme, just more quickly. However, the uncatalyzed, "spontaneous" reaction might lead to different products than the catalyzed reaction. Furthermore, enzymes can couple two or more reactions, so that a thermodynamically favorable reaction can be used to "drive" a thermodynamically unfavorable one. For example, the cleavage of the high-energy compound <a href="/wiki/Adenosine_triphosphate.html" title="Adenosine triphosphate">ATP</a> is often used to drive other energetically unfavorable chemical reactions.</p>
<p>Enzymes catalyze the forward and backward reactions equally. They do not alter the equilibrium itself, but only the speed at which it is reached. For example, <a href="/wiki/Carbonic_anhydrase.html" title="Carbonic anhydrase">carbonic anhydrase</a> catalyzes its reaction in either direction depending on the concentration of its reactants.</p>
<dl>
<dd><img class='tex' src="http://upload.wikimedia.org/math/c/8/c/c8c255d26e3f46da6bbacb1606142e6f.png" alt="\mathrm{CO_2 + H_2O {}^\mathrm{\quad Carbonic\ anhydrase} \!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\! \overrightarrow{\qquad\qquad\qquad\qquad} H_2CO_3}" /> (in <a href="/wiki/Biological_tissue.html" title="Biological tissue">tissues</a>; high CO<sub>2</sub> concentration)</dd>
<dd><img class='tex' src="http://upload.wikimedia.org/math/7/e/2/7e2012023ac3500d9e061834ffc2074e.png" alt="\mathrm{H_2CO_3 {}^\mathrm{\quad Carbonic\ anhydrase} \!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\!\! \overrightarrow{\qquad\qquad\qquad\qquad} CO_2 + H_2O}" /> (in <a href="/wiki/Lung.html" title="Lung">lungs</a>; low CO<sub>2</sub> concentration)</dd>
</dl>
<p>Nevertheless, if the physiological concentrations of the substrates and products have a large negative Gibbs free energy (<a href="/wiki/Exergonic.html" title="Exergonic">exergonic</a>), then the reaction is <i>effectively</i> irreversible. Under these conditions it is possible that the enzyme will only catalyze the reaction in one direction.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Kinetics">edit</a>]</div>
<p><a name="Kinetics" id="Kinetics"></a></p>
<h2>Kinetics</h2>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/Enzyme_kinetics.html" title="Enzyme kinetics">Enzyme kinetics</a></i></div>
</dd>
</dl>
<div class="thumb tright">
<div style="width:302px;"><a  class="internal" title="Mechanism for a single substrate enzyme catalyzed reaction. The enzyme (E) binds a substrate (S) and produces a product (P)."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/9/96/Simple_mechanism.svg/300px-Simple_mechanism.svg.png" alt="Mechanism for a single substrate enzyme catalyzed reaction. The enzyme (E) binds a substrate (S) and produces a product (P)." width="300" height="117" longdesc="/wiki/Image:Simple_mechanism.svg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Mechanism for a single substrate enzyme catalyzed reaction. The enzyme (E) binds a substrate (S) and produces a product (P).</div>
</div>
</div>
<p>Enzyme kinetics is the investigation of how enzymes bind substrates and turn them into products. The rate data used in kinetic analyses are obtained from <a href="/wiki/Enzyme_assay.html" title="Enzyme assay">enzyme assays</a>. In 1913 <a href="/wiki/Leonor_Michaelis.html" title="Leonor Michaelis">Leonor Michaelis</a> and <a href="/wiki/Maud_Menten.html" title="Maud Menten">Maud Menten</a> proposed a quantitative theory of enzyme kinetics, which is referred to as <a href="/wiki/Michaelis-Menten_kinetics.html" title="Michaelis-Menten kinetics">Michaelis-Menten kinetics</a>.<sup id="_ref-20" class="reference"><a  title="">[21]</a></sup> Their work was further developed by G. E. Briggs and <a href="/wiki/J._B._S._Haldane.html" title="J. B. S. Haldane">J. B. S. Haldane</a>, who derived kinetic equations that are still widely used today.<sup id="_ref-21" class="reference"><a  title="">[22]</a></sup></p>
<p>The major contribution of Michaelis and Menten was to think of enzyme reactions in two stages. In the first, the substrate binds reversibly to the enzyme, forming the enzyme-substrate complex. This is sometimes called the Michaelis-Menten complex in their honor. The enzyme then catalyzes the chemical step in the reaction and releases the product.</p>
<div class="thumb tright">
<div style="width:302px;"><a  class="internal" title="Saturation curve for an enzyme reaction showing the relation between the substrate concentration (S) and rate (v)."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/2/26/MM_curve.png/300px-MM_curve.png" alt="Saturation curve for an enzyme reaction showing the relation between the substrate concentration (S) and rate (v)." width="300" height="229" longdesc="/wiki/Image:MM_curve.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Saturation curve for an enzyme reaction showing the relation between the substrate concentration (S) and rate (<i>v</i>).</div>
</div>
</div>
<p>Enzymes can catalyze up to several million reactions per second. To find the maximum speed of an enzymatic reaction, the substrate concentration is increased until a constant rate of product formation is seen. This is shown in the saturation curve, shown on the right. Saturation happens because, as substrate concentration increases, more and more of the free enzyme is converted into the substrate-bound ES form. At the maximum velocity (<i>V</i><sub>max</sub>) of the enzyme, all enzyme active sites are saturated with substrate, and the amount of ES complex is the same as the total amount of enzyme.</p>
<p>However, <i>V</i><sub>max</sub> is only one kinetic constant of enzymes. The amount of substrate needed to achieve a given rate of reaction is also important. This is given by the <a href="/wiki/Michaelis-Menten_constant.html" title="Michaelis-Menten constant">Michaelis-Menten constant</a> (<i>K</i><sub>m</sub>), which is the substrate concentration required for an enzyme to reach one-half its maximum velocity. Each enzyme has a characteristic <i>K</i><sub>m</sub> for a given substrate, and this can show how tight the binding of the substrate is to the enzyme. Another useful constant is <i>k</i><sub>cat</sub>, which is the number of substrate molecules handled by one active site per second.</p>
<p>The efficiency of an enzyme can be expressed in terms of <i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub>. This is also called the specificity constant and incorporates the <a href="/wiki/Rate_constant.html" title="Rate constant">rate constants</a> for all steps in the reaction. Because the specificity constant reflects both affinity and catalytic ability, it is useful for comparing different enzymes against each other, or the same enzyme with different substrates. The theoretical maximum for the specificity constant is called the diffusion limit and is about 10<sup>8</sup> to 10<sup>9</sup> (M<sup>-1</sup> s<sup>-1</sup>). At this point every collision of the enzyme with its substrate will result in catalysis, and the rate of product formation is not limited by the reaction rate but by the diffusion rate. Enzymes with this property are called <i><a href="/wiki/Catalytically_perfect_enzyme.html" title="Catalytically perfect enzyme">catalytically perfect</a></i> or <i>kinetically perfect</i>. Example of such enzymes are <a href="/wiki/Triosephosphateisomerase.html" title="Triosephosphateisomerase">triose-phosphate isomerase</a>, <a href="/wiki/Carbonic_anhydrase.html" title="Carbonic anhydrase">carbonic anhydrase</a>, <a href="/wiki/Acetylcholinesterase.html" title="Acetylcholinesterase">acetylcholinesterase</a>, <a href="/wiki/Catalase.html" title="Catalase">catalase</a>, fumarase, ß-lactamase, and <a href="/wiki/Superoxide_dismutase.html" title="Superoxide dismutase">superoxide dismutase</a>.</p>
<p>Some enzymes operate with kinetics which are faster than diffusion rates, which would seem to be impossible. Several mechanisms have been invoked to explain this phenomenon. Some proteins are believed to accelerate catalysis by drawing their substrate in and pre-orienting them by using dipolar electric fields. Other models invoke a quantum-mechanical <a href="/wiki/Quantum_tunneling.html" title="Quantum tunneling">tunneling</a> explanation whereby a proton or an electron can tunnel through activation barriers, although for protons tunneling remains somewhat controversial.<sup id="_ref-22" class="reference"><a  title="">[23]</a></sup><sup id="_ref-23" class="reference"><a  title="">[24]</a></sup></p>
<p>Quantum tunneling for protons has been observed in <a href="/wiki/Tryptamine.html" title="Tryptamine">tryptamine</a>.<sup id="_ref-24" class="reference"><a  title="">[25]</a></sup> This suggests "through the barrier" may often be a more accurate characterization of enzyme catalysis than the traditional "over the barrier" model.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Inhibition">edit</a>]</div>
<p><a name="Inhibition" id="Inhibition"></a></p>
<h2>Inhibition</h2>
<div class="thumb tright">
<div style="width:402px;"><a  class="internal" title="A competitive inhibitor binds reversibly to the enzyme, preventing the binding of substrate. On the other hand, binding of substrate prevents binding of the inhibitor. Substrate and inhibitor compete for the enzyme."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/5/54/Competitive_inhibition_copy.svg/400px-Competitive_inhibition_copy.svg.png" alt="A competitive inhibitor binds reversibly to the enzyme, preventing the binding of substrate. On the other hand, binding of substrate prevents binding of the inhibitor. Substrate and inhibitor compete for the enzyme." width="400" height="280" longdesc="/wiki/Image:Competitive_inhibition_copy.svg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
A competitive inhibitor binds reversibly to the enzyme, preventing the binding of substrate. On the other hand, binding of substrate prevents binding of the inhibitor. Substrate and inhibitor compete for the enzyme.</div>
</div>
</div>
<div class="thumb tright">
<div style="width:402px;"><a  class="internal" title="The coenzyme folic acid (left) and the anti-cancer drug methotrexate (right) are very similar in structure. As a result, methotrexate is a competitive inhibitor of many enzymes that use folates."><img src="http://upload.wikimedia.org/wikipedia/en/6/67/Methotrexate_and_folic_acid_compared.png" alt="The coenzyme folic acid (left) and the anti-cancer drug methotrexate (right) are very similar in structure. As a result, methotrexate is a competitive inhibitor of many enzymes that use folates." width="400" height="128" longdesc="/wiki/Image:Methotrexate_and_folic_acid_compared.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
The coenzyme folic acid (left) and the anti-cancer drug methotrexate (right) are very similar in structure. As a result, methotrexate is a competitive inhibitor of many enzymes that use folates.</div>
</div>
</div>
<dl>
<dd>
<div class="noprint"><i>Main article: <a href="/wiki/Enzyme_inhibition.html" title="Enzyme inhibition">Enzyme inhibition</a></i></div>
</dd>
</dl>
<p>Enzymes reaction rates can be decreased by various types of <a href="/wiki/Enzyme_inhibitor.html" title="Enzyme inhibitor">enzyme inhibitors</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Reversible inhibitors">edit</a>]</div>
<p><a name="Reversible_inhibitors" id="Reversible_inhibitors"></a></p>
<h3>Reversible inhibitors</h3>
<p><b>Competitive inhibition</b></p>
<p>In competitive inhibition the inhibitor binds to the substrate binding site as shown (<i>right</i> top), thus preventing substrate from binding. Often competitive inhibitors strongly resemble the real substrate of the enzyme. For example, <a href="/wiki/Methotrexate.html" title="Methotrexate">methotrexate</a> is a competitive inhibitor of the enzyme <a href="/wiki/Dihydrofolate_reductase.html" title="Dihydrofolate reductase">dihydrofolate reductase</a>, which catalyzes the reduction of <a href="/wiki/Folic_acid.html" title="Folic acid">dihydrofolate</a> to <a href="/wiki/Folic_acid.html" title="Folic acid">tetrahydrofolate</a>. The similarity between the structures of folic acid and this drug are shown on the right.</p>
<p><b>Non-competitive inhibition</b></p>
<p>Non-competitive inhibitors never bind to the active site, but to other parts of the enzyme that can be far away from the substrate binding site (<i>right</i>, bottom). Their binding to this site changes the shape of the enzyme and stops the active site binding substrate(s). Consequently, since there is no direct competition between the substrate and inhibitor for the enzyme, the extent of inhibition depends only on the inhibitor concentration and will not be affected by the substrate concentration.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Irreversible inhibitors">edit</a>]</div>
<p><a name="Irreversible_inhibitors" id="Irreversible_inhibitors"></a></p>
<h3>Irreversible inhibitors</h3>
<p>Some enzyme inhibitors react with the enzyme and form a <a href="/wiki/Covalent.html" title="Covalent">covalent</a> adduct with the protein. The inactivation produced by this type of inhibitor cannot be reversed. A class of these compounds called <a href="/wiki/Suicide_inhibitor.html" title="Suicide inhibitor">Suicide inhibitors</a> includes <a href="/wiki/Eflornithine.html" title="Eflornithine">eflornithine</a> a drug used to treat the parasitic disease <a href="/wiki/Sleeping_sickness.html" title="Sleeping sickness">sleeping sickness</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Uses of inhibitors">edit</a>]</div>
<p><a name="Uses_of_inhibitors" id="Uses_of_inhibitors"></a></p>
<h3>Uses of inhibitors</h3>
<p>Inhibitors are often used as drugs, but they can also act as poisons. However, the difference between a drug and a poison is usually only a matter of amount, since most drugs are toxic at some level, as <a href="/wiki/Paracelsus.html" title="Paracelsus">Paracelsus</a> wrote, "<i>In all things there is a poison, and there is nothing without a poison.</i>"<sup id="_ref-25" class="reference"><a  title="">[26]</a></sup> Equally, <a href="/wiki/Antibiotics.html" title="Antibiotics">antibiotics</a> and other anti-infective drugs are just specific poisons that can kill a pathogen but not its host.</p>
<p>An example of an inhibitor being used as a drug is <a href="/wiki/Aspirin.html" title="Aspirin">aspirin</a>, which inhibits the <a href="/wiki/Cyclooxygenase.html" title="Cyclooxygenase">COX-1</a> and <a href="/wiki/Cyclooxygenase.html" title="Cyclooxygenase">COX-2</a> enzymes that produce the <a href="/wiki/Inflammation.html" title="Inflammation">inflammation</a> messenger <a href="/wiki/Prostaglandin.html" title="Prostaglandin">prostaglandin</a>, thus suppressing pain and inflammation. The poison <a href="/wiki/Cyanide.html" title="Cyanide">cyanide</a> is an irreversible enzyme inhibitor that combines with the <a href="/wiki/Copper.html" title="Copper">copper</a> prosthetic groups of the enzyme <a href="/wiki/Cytochrome_c_oxidase.html" title="Cytochrome c oxidase">cytochrome c oxidase</a> and blocks <a href="/wiki/Cellular_respiration.html" title="Cellular respiration">cellular respiration</a>.</p>
<p>In many organisms inhibitors may act as part of a <a href="/wiki/Feedback.html" title="Feedback">feedback</a> mechanism. If an enzyme produces too much of one substance in the organism, that substance may act as an inhibitor for the enzyme that produces it, causing production of the substance to slow down or stop when there is sufficient amount. This is a form of <a href="/wiki/Negative_feedback.html" title="Negative feedback">negative feedback</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Biological function">edit</a>]</div>
<p><a name="Biological_function" id="Biological_function"></a></p>
<h2>Biological function</h2>
<p>Enzymes serve a wide variety of functions inside living organisms. They are indispensable for <a href="/wiki/Signal_transduction.html" title="Signal transduction">signal transduction</a> and cell regulation, often via <a href="/wiki/Kinase.html" title="Kinase">kinases</a> and <a href="/wiki/Phosphatase.html" title="Phosphatase">phosphatases</a>. They also generate movement, with <a href="/wiki/Myosin.html" title="Myosin">myosin</a> hydrolysing ATP to generate <a href="/wiki/Muscle_contraction.html" title="Muscle contraction">muscle contraction</a> and also moving cargo around the cell as part of the <a href="/wiki/Cytoskeleton.html" title="Cytoskeleton">cytoskeleton</a>. Other ATPases in the cell membrane are <a href="/wiki/Ion_pump_%28biology%29.html" title="Ion pump (biology)">ion pumps</a> involved in <a href="/wiki/Active_transport.html" title="Active transport">active transport</a>. Enzymes are also involved in more exotic functions, such as <a href="/wiki/Luciferase.html" title="Luciferase">luciferase</a> generating light in <a href="/wiki/Firefly.html" title="Firefly">fireflies</a>.</p>
<p><a href="/wiki/Virus.html" title="Virus">Viruses</a> can contain enzymes for infecting cells, such as the <a href="/wiki/HIV.html" title="HIV">HIV</a> <a href="/wiki/Integrase.html" title="Integrase">integrase</a> and <a href="/wiki/Reverse_transcriptase.html" title="Reverse transcriptase">reverse transcriptase</a>, or for viral release from cells, like the <a href="/wiki/Influenza.html" title="Influenza">influenza</a> virus <a href="/wiki/Neuraminidase.html" title="Neuraminidase">neuraminidase</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Metabolism">edit</a>]</div>
<p><a name="Metabolism" id="Metabolism"></a></p>
<h3>Metabolism</h3>
<p>Several enzymes can work together in a specific order, creating <a href="/wiki/Metabolic_pathway.html" title="Metabolic pathway">metabolic pathways</a>. In a metabolic pathway, one enzyme takes the product of another enzyme as a substrate. After the catalytic reaction, the product is then passed on to another enzyme. Sometimes more than one enzyme can catalyse the same reaction in parallel, this can allow more complex regulation: with for example a low contant activity being provided by one enzyme but an inducible high activity from a second enzyme.</p>
<p>Enzymes determine what steps occur in these pathways. Without enzymes, metabolism would neither progress through the same steps, nor be fast enough to serve the needs of the cell. Indeed, a metabolic pathway such as <a href="/wiki/Glycolysis.html" title="Glycolysis">glycolysis</a> could not exist independently of enzymes. Glucose, for example, can react directly with ATP to become <a href="/wiki/Phosphorylation.html" title="Phosphorylation">phosphorylated</a> at one or more of its carbons. However, if <a href="/wiki/Hexokinase.html" title="Hexokinase">hexokinase</a> is present, <a href="/wiki/Glucose-6-phosphate.html" title="Glucose-6-phosphate">glucose-6-phosphate</a> is the only product. Consequently, the network of metabolic pathways within each cell depends on the set of functional enzymes that are present.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Control of activity">edit</a>]</div>
<p><a name="Control_of_activity" id="Control_of_activity"></a></p>
<h2>Control of activity</h2>
<p>There are four main ways that enzyme activity is controlled in the cell.</p>
<ol>
<li>Enzyme production (<a href="/wiki/Transcription_%28genetics%29.html" title="Transcription (genetics)">transcription</a> and <a href="/wiki/Translation_%28genetics%29.html" title="Translation (genetics)">translation</a> of enzyme genes) can be enhanced or diminished by a cell in response to changes in the cell's environment. This form of <a href="/wiki/Regulation_of_gene_expression.html" title="Regulation of gene expression">gene regulation</a> is called <a href="/wiki/Enzyme_induction_and_inhibition.html" title="Enzyme induction and inhibition">enzyme induction and inhibition</a>. For example, bacteria may become <a href="/wiki/Antibiotic_resistance.html" title="Antibiotic resistance">resistant to antibiotics</a> such as <a href="/wiki/Penicillin.html" title="Penicillin">penicillin</a> because enzymes called <a href="/wiki/Beta-lactamase.html" title="Beta-lactamase">beta-lactamases</a> are induced that hydrolyse the crucial <a href="/wiki/Beta-lactam.html" title="Beta-lactam">beta-lactam ring</a> within the penicillin molecule. Another example are enzymes in the <a href="/wiki/Liver.html" title="Liver">liver</a> called <a href="/wiki/Cytochrome_P450_oxidase.html" title="Cytochrome P450 oxidase">cytochrome P450 oxidases</a>, which are important in <a href="/wiki/Drug_metabolism.html" title="Drug metabolism">drug metabolism</a>. Induction or inhibition of these enzymes can cause <a href="/wiki/Drug_interaction.html" title="Drug interaction">drug interactions</a>.</li>
<li>Enzymes can be compartmentalized, with different metabolic pathways occurring in different <a href="/wiki/Cellular_compartment.html" title="Cellular compartment">cellular compartments</a>. For example, <a href="/wiki/Fatty_acids.html" title="Fatty acids">fatty acids</a> are synthesized by one set of enzymes in the <a href="/wiki/Cytosol.html" title="Cytosol">cytosol</a>, <a href="/wiki/Endoplasmic_reticulum.html" title="Endoplasmic reticulum">endoplasmic reticulum</a> and the <a href="/wiki/Golgi_apparatus.html" title="Golgi apparatus">Golgi apparatus</a> and used by a different set of enzymes as a source of energy in the <a href="/wiki/Mitochondrion.html" title="Mitochondrion">mitochondrion</a>, through <a href="/wiki/%CE%92-oxidation.html" title="Β-oxidation">β-oxidation</a>.<sup id="_ref-26" class="reference"><a  title="">[27]</a></sup></li>
<li>Enzymes can be regulated by <a href="/wiki/Enzyme_inhibitor.html" title="Enzyme inhibitor">inhibitors</a> and activators. For example, the end product(s) of a metabolic pathway are often inhibitors for one of the first enzymes of the pathway (usually the first irreversible step, called <i>committed step</i>), thus regulating the amount of end product made by the pathways. Such a regulatory mechanism is called a <a href="/wiki/Negative_feedback.html" title="Negative feedback">negative feedback mechanism</a>, because the amount of the end product produced is regulated by its own concentration. Negative feedback mechanism can effectively adjust the rate of synthesis of intermediate metabolites according to the demands of the cells. This helps with effective allocations of materials and energy economy, and it prevents the excess manufacture of end products. Like other <a href="/wiki/Homeostasis.html" title="Homeostasis">homeostatic devices</a>, the control of enzymatic action helps to maintain a stable internal environment in living organisms.</li>
<li>Enzymes can be regulated through <a href="/wiki/Post-translational_modification.html" title="Post-translational modification">post-translational modification</a>. This can include <a href="/wiki/Phosphorylation.html" title="Phosphorylation">phosphorylation</a>, <a href="/wiki/Myristic_acid.html" title="Myristic acid">myristoylation</a> and <a href="/wiki/Glycosylation.html" title="Glycosylation">glycosylation</a>. For example, in the response to <a href="/wiki/Insulin.html" title="Insulin">insulin</a>, the <a href="/wiki/Phosphorylation.html" title="Phosphorylation">phosphorylation</a> of multiple enzymes, including <a href="/wiki/Glycogen_synthase.html" title="Glycogen synthase">glycogen synthase</a>, helps control the synthesis or degradation of <a href="/wiki/Glycogen.html" title="Glycogen">glycogen</a> and allows the cell to respond to changes in <a href="/wiki/Blood_sugar.html" title="Blood sugar">blood sugar</a>.<sup id="_ref-27" class="reference"><a  title="">[28]</a></sup> Another example of post-translational modification is the cleavage of the polypeptide chain. <a href="/wiki/Chymotrypsin.html" title="Chymotrypsin">Chymotrypsin</a>, a digestive <a href="/wiki/Protease.html" title="Protease">protease</a>, is produced in inactive form as <a href="/wiki/Chymotrypsinogen.html" title="Chymotrypsinogen">chymotrypsinogen</a> in the <a href="/wiki/Pancreas.html" title="Pancreas">pancreas</a> and transported in this form to the <a href="/wiki/Stomach.html" title="Stomach">stomach</a> where it is activated. This stops the enzyme from digesting the pancreas or other tissues before it enters the gut. This type of inactive precursor to an enzyme is known as a <a href="/wiki/Zymogen.html" title="Zymogen">zymogen</a>.</li>
</ol>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Involvement in disease">edit</a>]</div>
<p><a name="Involvement_in_disease" id="Involvement_in_disease"></a></p>
<h2>Involvement in disease</h2>
<div class="thumb tright">
<div style="width:202px;"><a  class="internal" title="Phenylalanine hydroxylase. Created from PDB 1KW0 "><img src="http://upload.wikimedia.org/wikipedia/en/thumb/9/97/Phenylalanine_hydroxylase_brighter.jpg/200px-Phenylalanine_hydroxylase_brighter.jpg" alt="Phenylalanine hydroxylase. Created from PDB 1KW0 " width="200" height="210" longdesc="/wiki/Image:Phenylalanine_hydroxylase_brighter.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
<a href="/wiki/Phenylalanine_hydroxylase.html" title="Phenylalanine hydroxylase">Phenylalanine hydroxylase</a>. Created from <a  class="external text" title="http://www.rcsb.org/pdb/explore.do?structureId=1KW0">PDB 1KW0</a></div>
</div>
</div>
<p>Since the tight control of enzyme activity is essential for homeostasis, any malfunction (mutation, overproduction, underproduction or deletion) of a single critical enzyme can lead to a <a href="/wiki/Genetic_disease.html" title="Genetic disease">genetic disease</a>. The importance of enzymes is shown by the fact that a lethal illness can be caused by the malfunction of just one type of enzyme out of the thousands of types present in our bodies.</p>
<p>One example is the most common type of <a href="/wiki/Phenylketonuria.html" title="Phenylketonuria">phenylketonuria</a>. Mutation of this gene causes a single amino acid change in the enzyme <a href="/wiki/Phenylalanine_hydroxylase.html" title="Phenylalanine hydroxylase">phenylalanine hydroxylase</a>, which catalyzes the first step in the degradation of <a href="/wiki/Phenylalanine.html" title="Phenylalanine">phenylalanine</a>. The resulting build-up of phenylalanine and related products can lead to <a href="/wiki/Mental_retardation.html" title="Mental retardation">mental retardation</a> if the disease is untreated.<sup id="_ref-28" class="reference"><a  title="">[29]</a></sup></p>
<p>Another example is when <a href="/wiki/Germline_mutation.html" title="Germline mutation">germline mutations</a> in genes coding for <a href="/wiki/DNA_repair.html" title="DNA repair">DNA repair</a> enzymes cause hereditary cancer syndromes such as <a href="/wiki/Xeroderma_pigmentosum.html" title="Xeroderma pigmentosum">xeroderma pigmentosum</a>. Defects in these enzymes cause cancer since the body is less able to repair mutations in the genome. This causes a slow accumulation of mutations and results in the development of many types of cancer in the sufferer.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Naming conventions">edit</a>]</div>
<p><a name="Naming_conventions" id="Naming_conventions"></a></p>
<h2>Naming conventions</h2>
<p>An enzyme's name is a description of what it does, with the word ending in <i>-ase</i>. Examples are <a href="/wiki/Alcohol_dehydrogenase.html" title="Alcohol dehydrogenase">alcohol dehydrogenase</a> and <a href="/wiki/DNA_polymerase.html" title="DNA polymerase">DNA polymerase</a>. <a href="/wiki/Kinase.html" title="Kinase">Kinases</a> are enzymes that transfer <a href="/wiki/Phosphate.html" title="Phosphate">phosphate</a> groups. This results in different enzymes with the same function having the same basic name; they are therefore distinguished by other characteristics, such as their optimal <a href="/wiki/PH.html" title="PH">pH</a> (<a href="/wiki/Alkaline_phosphatase.html" title="Alkaline phosphatase">alkaline phosphatase</a>) or their location (membrane <a href="/wiki/ATPase.html" title="ATPase">ATPase</a>). Furthermore, the reversibility of chemical reactions means that the normal physiological direction of an enzyme reaction may not be seen under laboratory conditions. This can result in the same enzyme being identified with two different names: one coming from the laboratory identification and the other from its behavior in the cell. For instance, the enzyme formally known as <i>xylitol:NAD+ 2-oxidoreductase (D-xylulose-forming)</i> is more commonly referred to from the cellular viewpoint as <i>D-xylulose reductase</i>, since the function of the enzyme in the cell is actually the reverse of what is often seen under laboratory conditions.</p>
<p>The <a href="/wiki/International_Union_of_Biochemistry_and_Molecular_Biology.html" title="International Union of Biochemistry and Molecular Biology">International Union of Biochemistry and Molecular Biology</a> and the <a href="/wiki/International_Union_of_Pure_and_Applied_Chemistry.html" title="International Union of Pure and Applied Chemistry">International Union of Pure and Applied Chemistry</a> have developed a <a href="/wiki/Nomenclature.html" title="Nomenclature">nomenclature</a> for enzymes, the <a href="/wiki/EC_number.html" title="EC number">EC numbers</a>; each enzyme is described by a sequence of four numbers preceded by "EC". However, this is not a perfect solution, as enzymes from different species or even very similar enzymes in the same species may have identical EC numbers.</p>
<p>The first number broadly classifies the enzyme based on its mechanism:</p>
<p>The top-level classification is</p>
<ul>
<li>EC 1 <i><a href="/wiki/Oxidoreductase.html" title="Oxidoreductase">Oxidoreductases</a></i>: catalyze <a href="/wiki/Oxidation.html" title="Oxidation">oxidation</a>/reduction reactions</li>
<li>EC 2 <i><a href="/wiki/Transferase.html" title="Transferase">Transferases</a></i>: transfer a <a href="/wiki/Functional_group.html" title="Functional group">functional group</a> (<i>e.g.</i> a methyl or phosphate group)</li>
<li>EC 3 <i><a href="/wiki/Hydrolase.html" title="Hydrolase">Hydrolases</a></i>: catalyze the <a href="/wiki/Hydrolysis.html" title="Hydrolysis">hydrolysis</a> of various bonds</li>
<li>EC 4 <i><a href="/wiki/Lyase.html" title="Lyase">Lyases</a></i>: cleave various bonds by means other than hydrolysis and oxidation</li>
<li>EC 5 <i><a href="/wiki/Isomerase.html" title="Isomerase">Isomerases</a></i>: catalyze <a href="/wiki/Isomer.html" title="Isomer">isomerization</a> changes within a single molecule</li>
<li>EC 6 <i><a href="/wiki/Ligase.html" title="Ligase">Ligases</a></i>: join two molecules with <a href="/wiki/Covalent_bond.html" title="Covalent bond">covalent bonds</a></li>
</ul>
<p>The complete nomenclature can be browsed at <a  class="external free" title="http://www.chem.qmul.ac.uk/iubmb/enzyme/">http://www.chem.qmul.ac.uk/iubmb/enzyme/</a>.</p>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Industrial applications">edit</a>]</div>
<p><a name="Industrial_applications" id="Industrial_applications"></a></p>
<h2>Industrial applications</h2>
<p>Enzymes are used in the <a href="/wiki/Chemical_industry.html" title="Chemical industry">chemical industry</a> and other industrial applications when extremely specific catalysts are required. However, enzymes in general are limited in the number of reactions they have evolved to catalyse and also by their lack of stability in <a href="/wiki/Organic_solvent.html" title="Organic solvent">organic solvents</a> and at high temperatures. Consequently, <a href="/wiki/Protein_engineering.html" title="Protein engineering">protein engineering</a> is an active area of research and involves attempts to create new enzymes with novel properties, either through rational design or <i>in vitro</i> evolution.<sup id="_ref-29" class="reference"><a  title="">[30]</a></sup><sup id="_ref-30" class="reference"><a  title="">[31]</a></sup></p>
<table class="wikitable">
<tr>
<th>Application</th>
<th>Enzymes used</th>
<th>Uses</th>
<th>Notes and examples</th>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;" rowspan="4"><b><a href="/wiki/Detergent.html" title="Detergent">Biological detergent</a></b></td>
<td style="border-top: solid 3px #aaaaaa;">Primarily <a href="/wiki/Protease.html" title="Protease">proteases</a>, produced in an <a href="/wiki/Extracellular.html" title="Extracellular">extracellular</a> form from <a href="/wiki/Bacteria.html" title="Bacteria">bacteria</a></td>
<td style="border-top: solid 3px #aaaaaa;">Used for presoak conditions and direct liquid applications helping with removal of protein stains from clothes.</td>
<td style="border-top: solid 3px #aaaaaa;" rowspan="4">
<div class="center">
<div class="floatnone"><span><a  class="image" title=""><img src="http://upload.wikimedia.org/wikipedia/en/thumb/a/aa/Washingpowder.jpg/180px-Washingpowder.jpg" alt="" width="180" height="135" longdesc="/wiki/Image:Washingpowder.jpg" /></a></span></div>
</div>
</td>
</tr>
<tr>
<td><a href="/wiki/Amylase.html" title="Amylase">Amylase</a> enzymes</td>
<td>Detergents for machine dish washing to remove resistant starch residues.</td>
</tr>
<tr>
<td><a href="/wiki/Lipase.html" title="Lipase">Lipase</a> enzymes</td>
<td>Used to assist in the removal of fatty and oily stains.</td>
</tr>
<tr>
<td><a href="/wiki/Cellulase.html" title="Cellulase">Cellulase</a> enzymes</td>
<td>Used in biological fabric conditioners.</td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;" rowspan="2"><b><a href="/wiki/Baking.html" title="Baking">Baking industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;"><a href="/wiki/Fungus.html" title="Fungus">Fungal</a> alpha-amylase enzymes are normally inactivated at about 50 degrees Celsius, but are destroyed during the baking process.</td>
<td style="border-top: solid 3px #aaaaaa;">Catalyze breakdown of starch in the <a href="/wiki/Flour.html" title="Flour">flour</a> to sugar. Yeast action on sugar produces carbon dioxide. Used in production of white bread, buns, and rolls.</td>
<td style="border-top: solid 3px #aaaaaa;" rowspan="2">
<div class="center">
<div class="thumb tnone">
<div style="width:252px;"><a  class="internal" title="alpha-amylase catalyzes the release of sugar monomers from starch"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/4/45/Amylose.svg/250px-Amylose.svg.png" alt="alpha-amylase catalyzes the release of sugar monomers from starch" width="250" height="101" longdesc="/wiki/Image:Amylose.svg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
alpha-amylase catalyzes the release of sugar monomers from starch</div>
</div>
</div>
</div>
</td>
</tr>
<tr>
<td>Protease enzymes</td>
<td>Biscuit manufacturers use them to lower the protein level of flour.</td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;"><b><a href="/wiki/Baby_food.html" title="Baby food">Baby foods</a></b></td>
<td style="border-top: solid 3px #aaaaaa;"><a href="/wiki/Trypsin.html" title="Trypsin">Trypsin</a></td>
<td style="border-top: solid 3px #aaaaaa;">To predigest baby foods.</td>
<td style="border-top: solid 3px #aaaaaa;"></td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;" rowspan="6"><b><a href="/wiki/Brewing.html" title="Brewing">Brewing industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;">Enzymes from barley are released during the mashing stage of beer production.</td>
<td style="border-top: solid 3px #aaaaaa;">They degrade starch and proteins to produce simple sugar, amino acids and peptides that are used by yeast for fermentation.</td>
<td style="border-top: solid 3px #aaaaaa;" rowspan="6">
<div class="center">
<div class="thumb tnone">
<div style="width:182px;"><a  class="internal" title="Germinating barley used for malt."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/3/32/Sjb_whiskey_malt.jpg/180px-Sjb_whiskey_malt.jpg" alt="Germinating barley used for malt." width="180" height="135" longdesc="/wiki/Image:Sjb_whiskey_malt.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Germinating barley used for malt.</div>
</div>
</div>
</div>
</td>
</tr>
<tr>
<td>Industrially produced barley enzymes</td>
<td>Widely used in the brewing process to substitute for the natural enzymes found in barley.</td>
</tr>
<tr>
<td>Amylase, glucanases, proteases</td>
<td>Split polysaccharides and proteins in the <a href="/wiki/Malt.html" title="Malt">malt</a>.</td>
</tr>
<tr>
<td>Betaglucosidase</td>
<td>Improve the filtration characteristics.</td>
<td></td>
</tr>
<tr>
<td>Amyloglucosidase</td>
<td>Low-calorie <a href="/wiki/Beer.html" title="Beer">beer</a>.</td>
</tr>
<tr>
<td>Proteases</td>
<td>Remove cloudiness produced during storage of beers.</td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;"><b><a href="/wiki/Juice.html" title="Juice">Fruit juices</a></b></td>
<td style="border-top: solid 3px #aaaaaa;">Cellulases, pectinases</td>
<td style="border-top: solid 3px #aaaaaa;">Clarify fruit juices</td>
<td style="border-top: solid 3px #aaaaaa;"></td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;" rowspan="4"><b><a href="/wiki/Dairy.html" title="Dairy">Dairy industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;">Rennin, derived from the stomachs of young <a href="/wiki/Ruminant.html" title="Ruminant">ruminant animals</a> (like calves and lambs).</td>
<td style="border-top: solid 3px #aaaaaa;">Manufacture of cheese, used to hydrolyze protein.</td>
<td style="border-top: solid 3px #aaaaaa;" rowspan="4">
<div class="center">
<div class="thumb tnone">
<div style="width:182px;"><a  class="internal" title="Roquefort cheese"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/9/92/Roquefort_cheese.jpg/180px-Roquefort_cheese.jpg" alt="Roquefort cheese" width="180" height="144" longdesc="/wiki/Image:Roquefort_cheese.jpg" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Roquefort cheese</div>
</div>
</div>
</div>
</td>
</tr>
<tr>
<td>Microbially produced enzyme</td>
<td>Now finding increasing use in the dairy industry.</td>
</tr>
<tr>
<td><a href="/wiki/Lipase.html" title="Lipase">Lipases</a></td>
<td>Is implemented during the production of <a href="/wiki/Roquefort_cheese.html" title="Roquefort cheese">Roquefort cheese</a> to enhance the ripening of the <a href="/wiki/Danish_Blue_cheese.html" title="Danish Blue cheese">blue-mould cheese</a>.</td>
</tr>
<tr>
<td>Lactases</td>
<td>Break down lactose to glucose and galactose.</td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;" rowspan="2"><b><a href="/wiki/Starch.html" title="Starch">Starch industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;">Amylases, amyloglucosideases and glucoamylases</td>
<td style="border-top: solid 3px #aaaaaa;">Converts starch into glucose and various <a href="/wiki/Inverted_sugar_syrup.html" title="Inverted sugar syrup">syrups</a>.</td>
<td style="border-top: solid 3px #aaaaaa;" rowspan="2">
<div class="center">
<div class="thumb tnone">
<div style="width:208px;">
<table style="background: transparent; border: 0; padding: 0; margin: 0;" cellspacing="0">
<tr>
<td style="padding: 0; margin: 0; border: 0;"><a  class="image" title="Glucose"><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/6/64/Glucose_Haworth.png/100px-Glucose_Haworth.png" alt="Glucose" width="100" height="79" longdesc="/wiki/Image:Glucose_Haworth.png" /></a></td>
<td style="padding: 0; margin: 0; border: 0; width: 2px"></td>
<td style="padding: 0; margin: 0; border: 0;"><a  class="image" title="Glucose"><img src="http://upload.wikimedia.org/wikipedia/en/thumb/a/a8/Alpha-D-Fructose-structure-corrected.png/100px-Alpha-D-Fructose-structure-corrected.png" alt="Glucose" width="100" height="64" longdesc="/wiki/Image:Alpha-D-Fructose-structure-corrected.png" /></a></td>
</tr>
<tr>
<td style="padding: 0; margin: 0; border: 0;">
<div class="thumbcaption">Glucose</div>
</td>
<td style="padding: 0; margin: 0; border: 0;"></td>
<td style="padding: 0; margin: 0; border: 0;">
<div class="thumbcaption">Fructose</div>
</td>
</tr>
</table>
</div>
</div>
</div>
</td>
</tr>
<tr>
<td>Glucose isomerase</td>
<td>Converts <a href="/wiki/Glucose.html" title="Glucose">glucose</a> into fructose (high fructose syrups derived from starchy materials have enhanced sweetening properties and lower <a href="/wiki/Calorie.html" title="Calorie">calorific values</a>).</td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;"><b><a href="/wiki/Rubber.html" title="Rubber">Rubber industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;"><a href="/wiki/Catalase.html" title="Catalase">Catalase</a></td>
<td style="border-top: solid 3px #aaaaaa;">To generate <a href="/wiki/Oxygen.html" title="Oxygen">oxygen</a> from <a href="/wiki/Peroxide.html" title="Peroxide">peroxide</a> to convert <a href="/wiki/Latex.html" title="Latex">latex</a> into foam rubber.</td>
<td style="border-top: solid 3px #aaaaaa;"></td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;"><b><a href="/wiki/Paper.html" title="Paper">Paper industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;"><a href="/wiki/Amylase.html" title="Amylase">Amylases</a></td>
<td style="border-top: solid 3px #aaaaaa;">Degrade starch to a lower <a href="/wiki/Viscosity.html" title="Viscosity">viscosity</a> product needed for sizing and coating paper</td>
<td style="border-top: solid 3px #aaaaaa;">
<div class="center">
<div class="thumb tnone">
<div style="width:182px;"><a  class="internal" title="The International Paper Company in Georgetown, South Carolina."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/c/ca/InternationalPaper6413.JPG/180px-InternationalPaper6413.JPG" alt="The International Paper Company in Georgetown, South Carolina." width="180" height="135" longdesc="/wiki/Image:InternationalPaper6413.JPG" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
The <a href="/wiki/International_Paper.html" title="International Paper">International Paper</a> Company in <a href="/wiki/Georgetown%2C_South_Carolina.html" title="Georgetown, South Carolina">Georgetown, South Carolina</a>.</div>
</div>
</div>
</div>
</td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;"><b><a href="/wiki/Photography.html" title="Photography">Photographic industry</a></b></td>
<td style="border-top: solid 3px #aaaaaa;">Protease (ficin)</td>
<td style="border-top: solid 3px #aaaaaa;">Dissolve <a href="/wiki/Gelatin.html" title="Gelatin">gelatin</a> off scrap <a href="/wiki/Photographic_film.html" title="Photographic film">film</a> allowing recovery of its <a href="/wiki/Silver.html" title="Silver">silver</a> content.</td>
<td style="border-top: solid 3px #aaaaaa;"></td>
</tr>
<tr>
<td style="border-top: solid 3px #aaaaaa;"><b><a href="/wiki/Molecular_biology.html" title="Molecular biology">Molecular biology</a></b></td>
<td style="border-top: solid 3px #aaaaaa;"><a href="/wiki/Restriction_enzyme.html" title="Restriction enzyme">Restriction enzymes</a>, <a href="/wiki/DNA_ligase.html" title="DNA ligase">DNA ligase</a> and <a href="/wiki/Polymerases.html" title="Polymerases">polymerases</a></td>
<td style="border-top: solid 3px #aaaaaa;">Used to manipulate DNA in <a href="/wiki/Genetic_engineering.html" title="Genetic engineering">genetic engineering</a>, important in <a href="/wiki/Pharmacology.html" title="Pharmacology">pharmacology</a>, <a href="/wiki/Agriculture.html" title="Agriculture">agriculture</a> and <a href="/wiki/Medicine.html" title="Medicine">medicine</a>. Essential for <a href="/wiki/Restriction_enzyme.html" title="Restriction enzyme">restriction digestion</a> and the <a href="/wiki/Polymerase_chain_reaction.html" title="Polymerase chain reaction">polymerase chain reaction</a>. Molecular biology is also important in <a href="/wiki/Forensic_science.html" title="Forensic science">forensic science</a>.</td>
<td style="border-top: solid 3px #aaaaaa;">
<div class="center">
<div class="thumb tnone">
<div style="width:182px;"><a  class="internal" title="Part of the DNA double helix."><img src="http://upload.wikimedia.org/wikipedia/en/thumb/d/d9/DNA123_rotated.png/180px-DNA123_rotated.png" alt="Part of the DNA double helix." width="180" height="100" longdesc="/wiki/Image:DNA123_rotated.png" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a  class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="Enlarge" /></a></div>
Part of the DNA <a href="/wiki/Double_helix.html" title="Double helix">double helix</a>.</div>
</div>
</div>
</div>
</td>
</tr>
</table>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: See also">edit</a>]</div>
<p><a name="See_also" id="See_also"></a></p>
<h2>See also</h2>
<ul>
<li><a href="/wiki/Enzyme_kinetics.html" title="Enzyme kinetics">Enzyme kinetics</a></li>
<li><a href="/wiki/Enzyme_inhibitor.html" title="Enzyme inhibitor">Enzyme inhibitor</a></li>
<li><a href="/wiki/Enzyme_assay.html" title="Enzyme assay">Enzyme assay</a></li>
</ul>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: References">edit</a>]</div>
<p><a name="References" id="References"></a></p>
<h2>References</h2>
<div class="references-small">
<ol class="references">
<li id="_note-0"><b><a  title="">^</a></b> <cite style="font-style:normal">Bairoch A. (2000). "<a  class="external text" title="http://www.expasy.org/NAR/enz00.pdf">The ENZYME database in 2000</a>". <i>Nucleic Acids Res</i> <b>28</b>: 304-305. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=10592255">PMID 10592255</a>.</cite></li>
<li id="_note-1"><b><a  title="">^</a></b> <cite style="font-style:normal">Radzicka A, Wolfenden R. (1995). "A proficient enzyme.". <i>Science</i> <b>6</b> (267): 90-931. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=7809611">PMID 7809611</a>.</cite></li>
<li id="_note-2"><b><a  title="">^</a></b> <a  class="external text" title="http://etext.lib.virginia.edu/toc/modeng/public/Wil4Sci.html">Williams, H. S. (1904) A History of Science: in Five Volumes. Volume IV: Modern Development of the Chemical and Biological Sciences Harper and Brothers (New York)</a></li>
<li id="_note-3"><b><a  title="">^</a></b> <cite style="font-style:normal">Dubos J. (1951). "Louis Pasteur: Free Lance of Science, Gollancz. Quoted in Manchester K. L. (1995) Louis Pasteur (1822–1895)--chance and the prepared mind.". <i>Trends Biotechnol</i> <b>13</b> (12): 511-515. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=8595136">PMID 8595136</a>.</cite></li>
<li id="_note-4"><b><a  title="">^</a></b> <a  class="external text" title="http://nobelprize.org/nobel_prizes/chemistry/laureates/1907/buchner-bio.html">Nobel Laureate Biography of Eduard Buchner at http://nobelprize.org</a></li>
<li id="_note-5"><b><a  title="">^</a></b> <a  class="external text" title="http://nobelprize.org/nobel_prizes/chemistry/laureates/1907/buchner-lecture.html">Text of Eduard Buchner's 1907 Nobel lecture at http://nobelprize.org</a></li>
<li id="_note-6"><b><a  title="">^</a></b> <a  class="external text" title="http://nobelprize.org/nobel_prizes/chemistry/laureates/1946/">1946 Nobel prize for Chemistry laureates at http://nobelprize.org</a></li>
<li id="_note-7"><b><a  title="">^</a></b> <cite style="font-style:normal">Blake CC, Koenig DF, Mair GA, North AC, Phillips DC, Sarma VR. (1965). "Structure of hen egg-white lysozyme. A three-dimensional Fourier synthesis at 2 Angstrom resolution.". <i>Nature</i> <b>22</b> (206): 757-761. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=5891407">PMID 5891407</a>.</cite></li>
<li id="_note-8"><b><a  title="">^</a></b> <cite style="font-style:normal">Anfinsen C.B. (1973). "Principles that Govern the Folding of Protein Chains". <i>Science</i>: 223-230. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=4124164">PMID 4124164</a>.</cite></li>
<li id="_note-9"><b><a  title="">^</a></b> <a  class="external text" title="http://www.ebi.ac.uk/thornton-srv/databases/CSA/">The Catalytic Site Atlas at The European Bioinformatics Institute</a></li>
<li id="_note-10"><b><a  title="">^</a></b> <cite style="font-style:normal">Jaeger KE, Eggert T. (2004). "Enantioselective biocatalysis optimized by directed evolution.". <i>Curr Opin Biotechnol.</i> <b>15(4)</b>: 305-313. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=15358000">PMID 15358000</a>.</cite></li>
<li id="_note-11"><b><a  title="">^</a></b> <cite style="font-style:normal">Shevelev IV, Hubscher U. (2002). "The 3' 5' exonucleases.". <i>Nat Rev Mol Cell Biol.</i> <b>3</b> (5): 364-376. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=11988770">PMID 11988770</a>.</cite></li>
<li id="_note-12"><b><a  title="">^</a></b> Berg J., Tymoczko J. and Stryer L. (2002) <i>Biochemistry.</i> W. H. Freeman and Company| <a  class="internal">ISBN 0-7167-4955-6</a></li>
<li id="_note-13"><b><a  title="">^</a></b> <cite style="font-style:normal">Ibba M, Soll D. (2000). "Aminoacyl-tRNA synthesis.". <i>Annu Rev Biochem.</i> <b>69</b>: 617-650. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=10966471">PMID 10966471</a>.</cite></li>
<li id="_note-14"><b><a  title="">^</a></b> <cite style="font-style:normal">Rodnina MV, Wintermeyer W. (2001). "Fidelity of aminoacyl-tRNA selection on the ribosome: kinetic and structural mechanisms.". <i>Annu Rev Biochem.</i> <b>70</b>: 415-435. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=11395413">PMID 11395413</a>.</cite></li>
<li id="_note-15"><b><a  title="">^</a></b> <cite style="font-style:normal">Fischer E. (1894). "Einfluss der Configuration auf die Wirkung der Enzyme". <i>Ber. Dt.</i> Chem. Ges. <i><b>27</b>: 2985-2993.</i></cite></li>
<li id="_note-16"><b><a  title="">^</a></b> <cite style="font-style:normal">Koshland D. E. (1958). "Application of a Theory of Enzyme Specificity to Protein Synthesis". <i>Proc. Natl. Acad. Sci.</i> <b>44</b> (2): 98-104. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=16590179">PMID 16590179</a>.</cite></li>
<li id="_note-17"><b><a  title="">^</a></b> <cite style="font-style:normal">Mitsuhashi S, Mizushima T, Yamashita E, Yamamoto M, Kumasaka T, Moriyama H, Ueki T, Miyachi S, Tsukihara T. (2000). "X-ray structure of beta-carbonic anhydrase from the red alga, Porphyridium purpureum, reveals a novel catalytic site for CO(2) hydration.". <i>J Biol Chem.</i> <b>275(8)</b>: 5521-5526. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=10681531">PMID 10681531</a>.</cite></li>
<li id="_note-18"><b><a  title="">^</a></b> AF Wagner, KA Folkers (1975) <i>Vitamins and coenzymes.</i> Interscience Publishers New York| <a  class="internal">ISBN 0-88275-258-8</a></li>
<li id="_note-19"><b><a  title="">^</a></b> <a  class="external text" title="http://www.brenda.uni-koeln.de/">BRENDA The Comprehensive Enzyme Information System</a></li>
<li id="_note-20"><b><a  title="">^</a></b> <cite style="font-style:normal">Michaelis L., Menten M. (1913). "Die Kinetik der Invertinwirkung". <i>Biochem. Z.</i> <b>49</b>: 333-369.</cite></li>
<li id="_note-21"><b><a  title="">^</a></b> <cite style="font-style:normal">Briggs G. E., Haldane J. B. S. (1925). "A note on the kinetics of enzyme action". <i>Biochem. J.</i> <b>19</b>: 339-339. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=16743508">PMID 16743508</a>.</cite></li>
<li id="_note-22"><b><a  title="">^</a></b> <cite style="font-style:normal">Garcia-Viloca M., Gao J., Karplus M., Truhlar D. G. (2004). "How enzymes work: analysis by modern rate theory and computer simulations.". <i>Science</i> <b>303</b> (5655): 186 - 195. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=14716003">PMID 14716003</a>.</cite></li>
<li id="_note-23"><b><a  title="">^</a></b> <cite style="font-style:normal">Olsson M. H., Siegbahn P. E., Warshel A. (2004). "Simulations of the large kinetic isotope effect and the temperature dependence of the hydrogen atom transfer in lipoxygenase". <i>J. Am. Chem. Soc.</i> <b>126</b> (9): 2820-1828. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=14995199">PMID 14995199</a>.</cite></li>
<li id="_note-24"><b><a  title="">^</a></b> <cite style="font-style:normal">Masgrau L., Roujeinikova A., Johannissen L. O., Hothi P., Basran J., Ranaghan K. E., Mulholland A. J., Sutcliffe M. J., Scrutton N. S., Leys D. (2006). "Atomic Description of an Enzyme Reaction Dominated by Proton Tunneling". <i>Science</i> <b>312</b> (5771): 237-241. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=16614214">PMID 16614214</a>.</cite></li>
<li id="_note-25"><b><a  title="">^</a></b> Ball, Philip (2006) <i>The Devil's Doctor: Paracelsus and the World of Renaissance Magic and Science.</i> Farrar, Straus and Giroux <a  class="internal">ISBN 0-374-22979-1</a></li>
<li id="_note-26"><b><a  title="">^</a></b> <cite style="font-style:normal">Faergeman N. J, Knudsen J. (1997). "Role of long-chain fatty acyl-CoA esters in the regulation of metabolism and in cell signalling". <i>Biochem J</i> <b>323</b>: 1-12. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=9173866">PMID 9173866</a>.</cite></li>
<li id="_note-27"><b><a  title="">^</a></b> <cite style="font-style:normal">Doble B. W., Woodgett J. R. (2003). "GSK-3: tricks of the trade for a multi-tasking kinase". <i>J. Cell. Sci.</i> <b>116</b>: 1175-1186. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=12615961">PMID 12615961</a>.</cite></li>
<li id="_note-28"><b><a  title="">^</a></b> <a  class="external text" title="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=gnd.section.234">Phenylketonuria: NCBI Genes and Disease</a></li>
<li id="_note-29"><b><a  title="">^</a></b> <cite style="font-style:normal">Renugopalakrishnan V, Garduno-Juarez R, Narasimhan G, Verma CS, Wei X, Li P. (2005). "Rational design of thermally stable proteins: relevance to bionanotechnology.". <i>J Nanosci Nanotechnol.</i> <b>5</b> (11): 1759-1767. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=16433409">PMID 16433409</a>.</cite></li>
<li id="_note-30"><b><a  title="">^</a></b> <cite style="font-style:normal">Hult K, Berglund P. (2003). "Engineered enzymes for improved organic synthesis.". <i>Curr Opin Biotechnol.</i> <b>14</b> (4): 395-400. <a  class="external" title="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=12943848">PMID 12943848</a>.</cite></li>
</ol>
</div>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: Further reading">edit</a>]</div>
<p><a name="Further_reading" id="Further_reading"></a></p>
<h2>Further reading</h2>
<p><b>Etymology and history</b></p>
<ul>
<li><a  class="external text" title="http://bip.cnrs-mrs.fr/bip10/buchner.htm">New Beer in an Old Bottle: Eduard Buchner and the Growth of Biochemical Knowledge, edited by Athel Cornish-Bowden and published by Universitat de València (1997): ISBN 84-370-3328-4</a>, A history of early enzymology.</li>
<li><a  class="external text" title="http://etext.lib.virginia.edu/toc/modeng/public/Wil4Sci.html">Williams, Henry Smith, 1863-1943. A History of Science: in Five Volumes. Volume IV: Modern Development of the Chemical and Biological Sciences</a>, A textbook from the 19th century.</li>
<li>Kleyn, J. and Hough J. The Microbiology of Brewing. <i>Annual Review of Microbiology</i> (1971) Vol. 25: 583-608</li>
</ul>
<p><b>Enzyme structure and mechanism</b></p>
<ul>
<li>Fersht, A. Structure and Mechanism in Protein Science&#160;: A Guide to Enzyme Catalysis and Protein Folding. W. H. Freeman, 1998 <a  class="internal">ISBN 0-71673-268-8</a></li>
<li>Walsh, C., Enzymatic Reaction Mechanisms. W. H. Freeman and Company. 1979. <a  class="internal">ISBN 0-71670-070-0</a></li>
<li>Page, M. I., and Williams, A. (Eds.), 1987. Enzyme Mechanisms. Royal Society of Chemistry. <a  class="internal">ISBN 0-85186-947-5</a></li>
<li>M.V. Volkenshtein, R.R. <a href="/wiki/Revaz_Dogonadze.html" title="Revaz Dogonadze">Dogonadze</a>, A.K. Madumarov, Z.D. Urushadze, Yu.I. Kharkats. Theory of Enzyme Catalysis.- <i>Molekuliarnaya Biologia</i>, (1972), 431-439 (In Russian, English summary)</li>
</ul>
<p><b>Thermodynamics</b></p>
<ul>
<li><a  class="external text" title="http://www.emc.maricopa.edu/faculty/farabee/BIOBK/BioBookEnzym.html">Reactions and Enzymes</a> Chapter 10 of On-Line Biology Book at Estrella Mountain Community College.</li>
</ul>
<p><b>Kinetics and Inhibition</b></p>
<ul>
<li>Athel Cornish-Bowden, <i>Fundamentals of Enzyme Kinetics</i>. (3rd edition), Portland Press (2004), <a  class="internal">ISBN 1-85578-158-1</a>.</li>
<li>Irwin H. Segel, <i>Enzyme Kinetics&#160;: Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems</i>. Wiley-Interscience; New Ed edition (1993), <a  class="internal">ISBN 0-47130-309-7</a>.</li>
</ul>
<p><b>Function and control of enzymes in the cell</b></p>
<ul>
<li>Price, N. and Stevens, L., Fundamentals of Enzymology: Cell and Molecular Biology of Catalytic Proteins Oxford University Press, (1999), <a  class="internal">ISBN 0-19850-229-X</a></li>
<li><a  class="external text" title="http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=gnd.chapter.86">Nutritional and Metabolic Diseases</a></li>
</ul>
<p><b>Enzyme-naming conventions</b></p>
<ul>
<li><a  class="external text" title="http://www.chem.qmul.ac.uk/iubmb/enzyme/">Enzyme Nomenclature</a>, Recommendations for enzyme names from the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology.</li>
<li>Koshland D. The Enzymes, v. I, ch. 7, Acad. Press, New York, (1959)</li>
</ul>
<p><b>Industrial Applications</b></p>
<ul>
<li><a  class="external text" title="http://www.mapsenzymes.com/History_of_Enzymes.asp">History of industrial enzymes</a>, Article about the history of industrial enzymes, from the late 1900's to the present times.</li>
</ul>
<div class="editsection" style="float:right;margin-left:5px;">[<a  title="Edit section: External links">edit</a>]</div>
<p><a name="External_links" id="External_links"></a></p>
<h2>External links</h2>
<div class="infobox sisterproject">
<div class="floatleft"><span><a  class="image" title=" "><img src="http://upload.wikimedia.org/wikipedia/commons/thumb/4/4a/Commons-logo.svg/45px-Commons-logo.svg.png" alt=" " width="45" height="60" longdesc="/wiki/Image:Commons-logo.svg" /></a></span></div>
<div style="margin-left: 60px;"><a href="/wiki/Wikimedia_Commons.html" title="Wikimedia Commons">Wikimedia Commons</a> has media related to:
<div style="margin-left: 10px;"><i><b><a  class="extiw" title="commons:Category:Enzymes">Category:Enzymes</a></b></i></div>
</div>
</div>
<ul>
<li><a  class="external text" title="http://tutor.lscf.ucsb.edu/instdev/sears/biochemistry/tw-enz/tabs-enzymes-frames.htm">Structure/Function of Enzymes</a>, Web tutorial on enzyme structure and function.</li>
<li><a  class="external text" title="http://www.amfep.org">AMFEP</a>, Association of Manufacturers and Formulators of Enzyme Products</li>
<li><a  class="external text" title="http://us.expasy.org/enzyme/">ExPASy enzyme database</a>, links to <a href="/wiki/Swiss-Prot.html" title="Swiss-Prot">Swiss-Prot</a> sequence data, entries in other databases and to related literature searches.</li>
<li><a  class="external text" title="http://www.ebi.ac.uk/thornton-srv/databases/enzymes/">Enzyme Structures Database</a> links to the known 3-D structure data of enzymes in the <a href="/wiki/Protein_Data_Bank.html" title="Protein Data Bank">Protein Data Bank</a>.</li>
<li><a  class="external text" title="http://www-mitchell.ch.cam.ac.uk/macie">MACiE</a>, database of enzyme reaction mechanisms.</li>
<li><a  class="external text" title="http://www.biocrawler.com/biocorp/index.php?words=enzyme*&amp;GO=GO">Enzyme Companies</a></li>
<li><a  class="external text" title="http://www.brenda.uni-koeln.de">BRENDA</a>, comprehensive compilation of information and literature references about all known enzymes; requires payment by commercial users.</li>
<li><a  class="external text" title="http://www.genome.jp/kegg/">KEGG: Kyoto Encyclopedia of Genes and Genomes</a> Graphical and hypertext-based information on biochemical pathways and enzymes.</li>
</ul>
<div style="right:10px; display:none;" class="metadata topicon" id="featured-star">
<div style="position: relative; width: 14px; height: 14px; overflow: hidden">
<div style="position: absolute; top: 0px; left: 0px; font-size: 100px; overflow: hidden; line-height: 100px; z-index: 3"><a  title="Wikipedia:Featured articles"><span title="This is a featured article. Click here for more information.">&#160;&#160;&#160;</span></a></div>
<div style="position: absolute; top: 0px; left: 0px; z-index: 2"><a  class="image" title="This is a featured article. Click here for more information."><img src="http://upload.wikimedia.org/wikipedia/en/6/60/LinkFA-star.png" alt="This is a featured article. Click here for more information." width="14" height="14" longdesc="/wiki/Image:LinkFA-star.png" /></a></div>
</div>
</div>
<p><span class="plainlinks" id="purgelink"><a  class="external text" title="http://en.wikipedia.org/w/index.php?title=Enzyme&amp;action=purge">Purge this page's server cache</a></span></p>
<p><span class="FA" id="bg" style="display:none;"></span> <span class="FA" id="de" style="display:none;"></span></p>

<!-- 
Pre-expand include size: 56321 bytes
Post-expand include size: 15861 bytes
Template argument size: 12661 bytes
Maximum: 2048000 bytes
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:9257-0!1!0!default!!en!2 and timestamp 20060910160515 -->
<div class="printfooter">
Retrieved from "<a </div>
			<div id="catlinks"><p class='catlinks'><a  title="Special:Categories">Categories</a>: <span dir='ltr'><a  title="Category:Biochemistry">Biochemistry</a></span> | <span dir='ltr'><a  title="Category:Enzymes">Enzymes</a></span> | <span dir='ltr'><a  title="Category:Metabolism">Metabolism</a></span></p></div>			<!-- end content -->
			<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<ul>
				 <li id="ca-nstab-main" class="selected"><a href="/wiki/Enzyme.html">Article</a></li>
				 <li id="ca-talk"><a >Discussion</a></li>
				 <li id="ca-edit"><a >Edit this page</a></li>
				 <li id="ca-history"><a >History</a></li>
		</ul>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-login"><a >Sign in / create account</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(/images/wiki-en.png);" href="/wiki/Main_Page.html" title="Main Page"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="/wiki/Main_Page.html">Main Page</a></li>
				<li id="n-portal"><a >Community Portal</a></li>
				<li id="n-Featured-articles"><a >Featured articles</a></li>
				<li id="n-currentevents"><a >Current events</a></li>
				<li id="n-recentchanges"><a >Recent changes</a></li>
				<li id="n-randompage"><a >Random article</a></li>
				<li id="n-help"><a >Help</a></li>
				<li id="n-contact"><a >Contact Wikipedia</a></li>
				<li id="n-sitesupport"><a >Donations</a></li>
			</ul>
		</div>
	</div>
		<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="/wiki/Special:Search" id="searchform"><div>
				<input id="searchInput" name="search" type="text" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" value="Search" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a >What links here</a></li>
				<li id="t-recentchangeslinked"><a >Related changes</a></li>
<li id="t-upload"><a >Upload file</a></li>
<li id="t-specialpages"><a >Special pages</a></li>
				<li id="t-print"><a >Printable version</a></li>				<li id="t-permalink"><a >Permanent link</a></li><li id="t-cite"><a >Cite this article</a></li>			</ul>
		</div>
	</div>
	<div id="p-lang" class="portlet">
		<h5>In other languages</h5>
		<div class="pBody">
			<ul>
				<li class="interwiki-ar"><a >العربية</a></li>
				<li class="interwiki-bn"><a >বাংলা</a></li>
				<li class="interwiki-zh-min-nan"><a >Bân-lâm-gú</a></li>
				<li class="interwiki-bs"><a >Bosanski</a></li>
				<li class="interwiki-bg"><a >Български</a></li>
				<li class="interwiki-ca"><a >Català</a></li>
				<li class="interwiki-cs"><a >Česky</a></li>
				<li class="interwiki-da"><a >Dansk</a></li>
				<li class="interwiki-de"><a >Deutsch</a></li>
				<li class="interwiki-et"><a >Eesti</a></li>
				<li class="interwiki-es"><a >Español</a></li>
				<li class="interwiki-eo"><a >Esperanto</a></li>
				<li class="interwiki-fa"><a >فارسی</a></li>
				<li class="interwiki-fr"><a >Français</a></li>
				<li class="interwiki-ko"><a >한국어</a></li>
				<li class="interwiki-hr"><a >Hrvatski</a></li>
				<li class="interwiki-io"><a >Ido</a></li>
				<li class="interwiki-id"><a >Bahasa Indonesia</a></li>
				<li class="interwiki-is"><a >Íslenska</a></li>
				<li class="interwiki-it"><a >Italiano</a></li>
				<li class="interwiki-he"><a >עברית</a></li>
				<li class="interwiki-la"><a >Latina</a></li>
				<li class="interwiki-lt"><a >Lietuvių</a></li>
				<li class="interwiki-hu"><a >Magyar</a></li>
				<li class="interwiki-mk"><a >Македонски</a></li>
				<li class="interwiki-ms"><a >Bahasa Melayu</a></li>
				<li class="interwiki-nl"><a >Nederlands</a></li>
				<li class="interwiki-ja"><a >日本語</a></li>
				<li class="interwiki-no"><a >Norsk (bokmål)</a></li>
				<li class="interwiki-ug"><a >Oyghurque</a></li>
				<li class="interwiki-pl"><a >Polski</a></li>
				<li class="interwiki-pt"><a >Português</a></li>
				<li class="interwiki-ru"><a >Русский</a></li>
				<li class="interwiki-simple"><a >Simple English</a></li>
				<li class="interwiki-sk"><a >Slovenčina</a></li>
				<li class="interwiki-sl"><a >Slovenščina</a></li>
				<li class="interwiki-sr"><a >Српски / Srpski</a></li>
				<li class="interwiki-su"><a >Basa Sunda</a></li>
				<li class="interwiki-fi"><a >Suomi</a></li>
				<li class="interwiki-sv"><a >Svenska</a></li>
				<li class="interwiki-th"><a >ไทย</a></li>
				<li class="interwiki-tr"><a >Türkçe</a></li>
				<li class="interwiki-uk"><a >Українська</a></li>
				<li class="interwiki-zh"><a >中文</a></li>
			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a ><img src="/skins-1.5/common/images/poweredby_mediawiki_88x31.png" alt="MediaWiki" /></a></div>
				<div id="f-copyrightico"><a ><img src="/images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>
			<ul id="f-list">
				<li id="lastmod"> This page was last modified 19:32, 9 September 2006.</li>
				<li id="copyright">All text is available under the terms of the <a class='internal'  title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal'  title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the Wikimedia Foundation, Inc.<br /></li>
				<li id="privacy"><a  title="wikimedia:Privacy policy">Privacy policy</a></li>
				<li id="about"><a  title="Wikipedia:About">About Wikipedia</a></li>
				<li id="disclaimer"><a  title="Wikipedia:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
		
	
		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
</div>
<!-- Served by srv34 in 0.136 secs. --></body></html>
